diff --git a/po/POTFILES.in b/po/POTFILES.in
index de9f491..d5a1409 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -1,273 +1,89 @@
-[encoding: UTF-8]
-./kolab/__init__.py
-./test-wallace-send.py
-./wallace/__init__.py
-./wallace/future_module_footer.py
-./wallace/future_module_correctsentdate.py
-./wallace/modules.py
-./wallace/future_module_googletranslate.py
-./wallace/future_module_dlp.py
-./wallace/module_optout.py
-./wallace/future_module_resources.py
-./wallace/future_module_freebusy.py
-./wallace/future_module_statistics.py
-./wallace/future_module_bcc.py
-./wallace/future_module_conversations.py
-./tests/__init__.py
-./tests/test-002-pykolab_conf_split.py
-./tests/test-003-conf_get_list.py
-./tests/test-001-import_pykolab_conf.py
-./tests/test-000-import_pykolab.py
-./setup-kolab.py
-./play/migrate_mail_to_uid_prod.py
-./play/detect-object-type.py
-./play/test_submission.py
-./play/test_undelete.py
-./play/kolab-sap/kolab_smtp_access_policy.py
-./play/confmgmt/__init__.py
-./play/confmgmt/augeas.py
-./play/confmgmt/db.py
-./play/confmgmt/model.py
-./play/test_folders.py
-./play/migrate_uid_to_mail.py
-./play/split_message_file.py
-./play/test-kolab-smtp-access-policy-load.py
-./play/test-login-as.py
-./play/test_imap.py
-./play/test.py
-./play/test_imapd.py
-./play/augeas-insert.py
-./play/imap_annotations_test.py
-./play/libkolabxml/todo.py
-./play/libkolabxml/event.py
-./play/libkolabxml/event_rfc822.py
-./play/libkolabxml/contact.py
-./play/libkolabxml/event_imap.py
-./play/persistent_search.py
-./play/migrate_mail_to_uid.py
-./play/noheaderini.py
-./play/flawed_zpush_testing_create_folders.py
-./play/migrate_uid_to_mail_prod.py
-./play/cleanup_acls.py
-./play/persistent_search_kolab_23.py
-./play/strip_many_headers.py
-./play/parallel_persistent_searches.py
-./play/sync_repl_kolab_23.py
-./play/test_socket.py
-./play/sqlalchemy_schemadisplay.py
-./play/test_filter.py
-./play/test_socket_client.py
-./play/test-parse_ldap_uri.py
-./play/roundcube_database.py
-./play/confmgmt.py
-./play/rolequota/__init__.py
-./play/test-output.py
-./play/test_cal_spread.py
-./play/migrate_lowercase_uid.py
-./play/test_kolabxml.py
-./play/unicode-to-ascii.py
-./play/openssl/license.py
-./play/fork.py
-./play/pygpgme/verify.py
-./play/pygpgme/sign.py
-./play/test-entitlements.py
-./play/unicode_test.py
-./play/load_test.py
-./play/conf.py
-./play/roundcube_database/__init__.py
-./play/roundcube_database/users.py
-./play/roundcube_database/identities.py
-./play/push_contacts.py
-./play/effectiverights.py
-./play/sync_client.py
-./play/anon-imap/anon-imap.py
-./play/test_page_control.py
-./play/wap/group.members_list.py
-./play/wap/user.info.py
-./play/wap/roles.list.py
-./play/wap/role.delete.py
-./play/wap/form_value.select_options-preferredlanguage.py
-./play/wap/form_value.list_options-c.py
-./play/wap/domain.info.py
-./play/wap/system.select_domain.py
-./play/wap/domains.capabilities.py
-./play/wap/role.find_by_attribute.py
-./play/wap/group.add.py
-./play/wap/form_value.generate_cn.py
-./play/wap/user_types.list.py
-./play/wap/form_value.generate_uid.py
-./play/wap/role.add.py
-./play/wap/scu.py
-./play/wap/user.delete.py
-./play/wap/group.info.py
-./play/wap/role.capabilities.py
-./play/wap/form_value.generate_password.py
-./play/wap/domains.list.py
-./play/wap/group_types.list.py
-./play/wap/user.edit.py
-./play/wap/form_value.generate_mail.py
-./play/wap/user.add.py
-./play/wap/form_value.select_options-ou.py
-./play/wap/groups.list.py
-./play/wap/form_value.generate_displayname.py
-./play/wap/users.list.py
-./play/wap/role.info.py
-./play/wap/system.capabilities.py
-./play/parse_policy.py
-./testaci.py
-./kwap/kwap/__init__.py
-./kwap/kwap/tests/__init__.py
-./kwap/kwap/tests/functional/__init__.py
-./kwap/kwap/tests/functional/test_authentication.py
-./kwap/kwap/tests/functional/test_root.py
-./kwap/kwap/tests/models/test_auth.py
-./kwap/kwap/tests/models/__init__.py
-./kwap/kwap/templates/__init__.py
-./kwap/kwap/templates/domain/__init__.py
-./kwap/kwap/controllers/__init__.py
-./kwap/kwap/controllers/template.py
-./kwap/kwap/controllers/secure.py
-./kwap/kwap/controllers/domain.py
-./kwap/kwap/controllers/error.py
-./kwap/kwap/controllers/root.py
-./kwap/kwap/model/__init__.py
-./kwap/kwap/model/auth.py
-./kwap/kwap/config/__init__.py
-./kwap/kwap/config/environment.py
-./kwap/kwap/config/app_cfg.py
-./kwap/kwap/config/middleware.py
-./kwap/kwap/websetup/__init__.py
-./kwap/kwap/websetup/schema.py
-./kwap/kwap/websetup/bootstrap.py
-./kwap/kwap/lib/__init__.py
-./kwap/kwap/lib/base.py
-./kwap/kwap/lib/app_globals.py
-./kwap/kwap/lib/helpers.py
-./kwap/kwap/grids/__init__.py
-./kwap/kwap/grids/domains.py
-./kwap/ez_setup/__init__.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/master.mak.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/error.mak.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/index.mak.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/environ.mak.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/about.mak.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/data.mak.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/login.mak.py
-./kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/authentication.mak.py
-./kwap/setup.py
-./saslauthd.py
-./kolabtest.py
-./test-urllib.py
-./bonnie/__init__.py
-./kolabd.py
-./kolabd/__init__.py
-./kolabd/process.py
-./ext/python/Tools/freeze/bkfile.py
-./ext/python/Tools/freeze/freeze.py
-./ext/python/Tools/freeze/makefreeze.py
-./ext/python/Tools/freeze/checkextensions.py
-./ext/python/Tools/freeze/winmakemakefile.py
-./ext/python/Tools/freeze/makemakefile.py
-./ext/python/Tools/freeze/makeconfig.py
-./ext/python/Tools/freeze/parsesetup.py
-./ext/python/Tools/freeze/checkextensions_win32.py
-./ext/python/Tools/freeze/hello.py
-./pykolab/__init__.py
-./pykolab/tests/tests.py
-./pykolab/tests/__init__.py
-./pykolab/tests/imap/__init__.py
-./pykolab/tests/imap/test_create_mailbox.py
-./pykolab/tests/imap/test_login_admin.py
-./pykolab/tests/imap/test_login.py
-./pykolab/tests/wap/__init__.py
-./pykolab/tests/wap/test_login.py
-./pykolab/auth/__init__.py
-./pykolab/auth/ldap/__init__.py
-./pykolab/auth/ldap/openldap/__init__.py
-./pykolab/auth/ldap/cache.py
-./pykolab/auth/ldap/rhds/__init__.py
-./pykolab/auth/ldap/syncrepl.py
-./pykolab/auth/ldap/msds/__init__.py
-./pykolab/auth/ldap/sunds/__init__.py
-./pykolab/auth/ldap/fds/__init__.py
-./pykolab/auth/sql/__init__.py
-./pykolab/utils.py
-./pykolab/base.py
-./pykolab/conf/__init__.py
-./pykolab/conf/defaults.py
-./pykolab/conf/parser.py
-./pykolab/conf/entitlement.py
-./pykolab/logger.py
-./pykolab/errors.py
-./pykolab/constants.py
-./pykolab/translate.py
-./pykolab/cli/__init__.py
-./pykolab/cli/telemetry/__init__.py
-./pykolab/cli/telemetry/cmd_list_sessions.py
-./pykolab/cli/telemetry/cmd_examine_session.py
-./pykolab/cli/telemetry/cmd_examine_command_issue.py
-./pykolab/cli/telemetry/cmd_expire_sessions.py
-./pykolab/cli/cmd_set_mail.py
-./pykolab/cli/commands.py
-./pykolab/cli/cmd_delete_mailbox.py
-./pykolab/cli/cmd_list_deleted_mailboxes.py
-./pykolab/cli/cmd_set_language.py
-./pykolab/cli/cmd_list_mailboxes.py
-./pykolab/cli/cmd_export_mailbox.py
-./pykolab/cli/cmd_transfer_mailbox.py
-./pykolab/cli/cmd_undelete_mailbox.py
-./pykolab/cli/cmd_sync.py
-./pykolab/cli/cmd_list_domains.py
-./pykolab/cli/cmd_remove_mailaddress.py
-./pykolab/cli/cmd_create_mailbox.py
-./pykolab/plugins/__init__.py
-./pykolab/plugins/defaultfolders/__init__.py
-./pykolab/plugins/dynamicquota/__init__.py
-./pykolab/plugins/recipientpolicy/__init__.py
-./pykolab/wap_client.old/__init__.py
-./pykolab/xml/__init__.py
-./pykolab/xml/2_0/__init__.py
-./pykolab/xml/event.py
-./pykolab/xml/common.py
-./pykolab/xml/1_0/__init__.py
-./pykolab/wap_client/__init__.py
-./pykolab/wap_client/request.py
-./pykolab/wap_client/connect.py
-./pykolab/setup/__init__.py
-./pykolab/setup/setup_zpush.py
-./pykolab/setup/setup_ldap.py
-./pykolab/setup/setup_roundcube.py
-./pykolab/setup/components.py
-./pykolab/setup/setup_mta.py
-./pykolab/setup/setup_imap.py
-./pykolab/setup/setup_kolabd.py
-./pykolab/setup/setup_mysql.py
-./pykolab/setup/setup_freebusy.py
-./pykolab/imap/__init__.py
-./pykolab/imap/dovecot.py
-./pykolab/imap/cyrus.py
-./pykolab/telemetry.py
-./pykolab/tests.old/mail.py
-./pykolab/tests.old/calendar.py
-./pykolab/tests.old/zpush/__init__.py
-./pykolab/tests.old/zpush/test_000_001.py
-./pykolab/tests.old/zpush/test_000_002.py
-./pykolab/tests.old/zpush/test_000_000.py
-./pykolab/tests.old/constants.py
-./pykolab/tests.old/contacts.py
-./pykolab/tests.old/ksap/__init__.py
-./pykolab/tests.old/ksap/test_sendera.py
-./pykolab/tests.old/create-contacts.py
-./wallace.py
-./test-wallace-loadmsg.py
-./kolab.py
-./cyruslib.py
-./test-wallace.py
-./test-search.py
-./conf.py
-./bin/kolab_smtp_access_policy.py
-./bin/kolab_parse_telemetry.py
-./bin/test-read-input.py
-./bin/test_parse_policy.py
-./bonnie.py
-./saslauthd/__init__.py
+bin/kolab_parse_telemetry.py
+bin/kolab_smtp_access_policy.py
+conf.py
+cyruslib.py
+ext/python/Tools/freeze/bkfile.py
+ext/python/Tools/freeze/checkextensions.py
+ext/python/Tools/freeze/checkextensions_win32.py
+ext/python/Tools/freeze/freeze.py
+ext/python/Tools/freeze/hello.py
+ext/python/Tools/freeze/makeconfig.py
+ext/python/Tools/freeze/makefreeze.py
+ext/python/Tools/freeze/makemakefile.py
+ext/python/Tools/freeze/parsesetup.py
+ext/python/Tools/freeze/winmakemakefile.py
+kolabd/__init__.py
+kolabd/process.py
+kolabd.py
+kolab/__init__.py
+kolab.py
+kolabtest.py
+pykolab/auth/__init__.py
+pykolab/auth/ldap/cache.py
+pykolab/auth/ldap/__init__.py
+pykolab/base.py
+pykolab/cli/cmd_add_domain.py
+pykolab/cli/cmd_create_mailbox.py
+pykolab/cli/cmd_delete_mailbox.py
+pykolab/cli/cmd_export_mailbox.py
+pykolab/cli/cmd_list_deleted_mailboxes.py
+pykolab/cli/cmd_list_domains.py
+pykolab/cli/cmd_list_mailboxes.py
+pykolab/cli/cmd_remove_mailaddress.py
+pykolab/cli/cmd_set_language.py
+pykolab/cli/cmd_set_mail.py
+pykolab/cli/cmd_sync.py
+pykolab/cli/cmd_transfer_mailbox.py
+pykolab/cli/cmd_undelete_mailbox.py
+pykolab/cli/commands.py
+pykolab/cli/__init__.py
+pykolab/cli/telemetry/cmd_examine_command_issue.py
+pykolab/cli/telemetry/cmd_examine_session.py
+pykolab/cli/telemetry/cmd_expire_sessions.py
+pykolab/cli/telemetry/cmd_list_sessions.py
+pykolab/cli/telemetry/__init__.py
+pykolab/conf/defaults.py
+pykolab/conf/entitlement.py
+pykolab/conf/__init__.py
+pykolab/constants.py
+pykolab/errors.py
+pykolab/imap/cyrus.py
+pykolab/imap/__init__.py
+pykolab/__init__.py
+pykolab/logger.py
+pykolab/plugins/defaultfolders/__init__.py
+pykolab/plugins/dynamicquota/__init__.py
+pykolab/plugins/__init__.py
+pykolab/plugins/recipientpolicy/__init__.py
+pykolab/setup/components.py
+pykolab/setup/__init__.py
+pykolab/setup/setup_freebusy.py
+pykolab/setup/setup_imap.py
+pykolab/setup/setup_kolabd.py
+pykolab/setup/setup_ldap.py
+pykolab/setup/setup_mta.py
+pykolab/setup/setup_mysql.py
+pykolab/setup/setup_roundcube.py
+pykolab/setup/setup_zpush.py
+pykolab/telemetry.py
+pykolab/tests/imap/__init__.py
+pykolab/tests/imap/test_create_mailbox.py
+pykolab/tests/imap/test_login_admin.py
+pykolab/tests/imap/test_login.py
+pykolab/tests/__init__.py
+pykolab/tests/tests.py
+pykolab/tests/wap/__init__.py
+pykolab/tests/wap/test_login.py
+pykolab/translate.py
+pykolab/utils.py
+pykolab/wap_client/connect.py
+pykolab/wap_client/__init__.py
+pykolab/wap_client/request.py
+saslauthd/__init__.py
+saslauthd.py
+setup-kolab.py
+test-wallace.py
+wallace/__init__.py
+wallace/module_optout.py
+wallace/modules.py
+wallace.py
diff --git a/po/pykolab.pot b/po/pykolab.pot
index 0d5af07..da8ffa4 100644
--- a/po/pykolab.pot
+++ b/po/pykolab.pot
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2012-05-09 14:27+0100\n"
+"POT-Creation-Date: 2012-08-22 16:47+0200\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
@@ -17,1863 +17,1923 @@ msgstr ""
 "Content-Type: text/plain; charset=CHARSET\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: .././wallace/__init__.py:37 .././bonnie/__init__.py:35
-#: .././kolabd/__init__.py:46 .././saslauthd/__init__.py:47
-msgid "Daemon Options"
-msgstr ""
-
-#: .././wallace/__init__.py:44 .././bonnie/__init__.py:41
-#: .././kolabd/__init__.py:52 .././saslauthd/__init__.py:53
-msgid "Fork to the background."
-msgstr ""
-
-#: .././wallace/__init__.py:52
-msgid "Bind address for Wallace."
+#: ../bin/kolab_smtp_access_policy.py:206
+#, python-format
+msgid "Adding policy request to instance %s"
 msgstr ""
 
-#: .././wallace/__init__.py:60 .././kolabd/__init__.py:58
-#: .././saslauthd/__init__.py:59
-msgid "Path to the PID file to use."
+#: ../bin/kolab_smtp_access_policy.py:446
+msgid "Unauthorized access not allowed"
 msgstr ""
 
-#: .././wallace/__init__.py:68
-msgid "Port that Wallace is supposed to use."
+#: ../bin/kolab_smtp_access_policy.py:467
+#: ../bin/kolab_smtp_access_policy.py:658
+msgid "Could not find recipient"
 msgstr ""
 
-#: .././wallace/__init__.py:102 .././wallace/__init__.py:126
-#: .././wallace/__init__.py:373 .././wallace/__init__.py:401
+#: ../bin/kolab_smtp_access_policy.py:486
+#: ../bin/kolab_smtp_access_policy.py:586
 #, python-format
-msgid "Number of threads currently running: %d"
+msgid "Could not find envelope sender user %s"
 msgstr ""
 
-#: .././wallace/__init__.py:111 .././wallace/__init__.py:135
+#: ../bin/kolab_smtp_access_policy.py:529
 #, python-format
-msgid "Continuing with %d threads currently running"
+msgid "Obtained authenticated user details for %r: %r"
 msgstr ""
 
-#. TODO: Apply throttling
-#: .././wallace/__init__.py:118
+#: ../bin/kolab_smtp_access_policy.py:608
 #, python-format
-msgid "Creating thread for message in %s"
+msgid "%s is unauthorized to send on behalf of %s"
 msgstr ""
 
-#: .././wallace/__init__.py:142
+#: ../bin/kolab_smtp_access_policy.py:618
 #, python-format
-msgid "Running thread %s for message file %s"
+msgid "User %s attempted to use envelope sender address %s "
 msgstr ""
 
-#: .././wallace/__init__.py:151
+#: ../bin/kolab_smtp_access_policy.py:682
+#: ../bin/kolab_smtp_access_policy.py:693
 #, python-format
-msgid "This message was already in module %s, delegating "
+msgid "Found user %s to be a delegate user of %s"
 msgstr ""
 
-#: .././wallace/__init__.py:160
+#: ../bin/kolab_smtp_access_policy.py:717
 #, python-format
-msgid "It was also in a certain stage: %s, letting "
+msgid "Verifying authenticated sender '%(sender)s' with "
 msgstr ""
 
-#: .././wallace/__init__.py:168 .././wallace/__init__.py:174
-#: .././wallace/__init__.py:184
+#: ../bin/kolab_smtp_access_policy.py:722
 #, python-format
-msgid "Executing module %s"
-msgstr ""
-
-#: .././wallace/__init__.py:211 .././bonnie/__init__.py:67
-#: .././kolabd/__init__.py:85 .././saslauthd/__init__.py:90
-msgid "Interrupted by user"
-msgstr ""
-
-#: .././wallace/__init__.py:215 .././wallace/__init__.py:225
-#: .././kolabd/__init__.py:90 .././kolabd/__init__.py:101
-msgid "Traceback occurred, please report a "
+msgid "Verifying unauthenticated sender '%(sender)s' "
 msgstr ""
 
-#: .././wallace/__init__.py:221 .././bonnie/__init__.py:75
-#: .././kolabd/__init__.py:96 .././saslauthd/__init__.py:98
+#: ../bin/kolab_smtp_access_policy.py:739
 #, python-format
-msgid "Type Error: %s"
-msgstr ""
-
-#: .././wallace/__init__.py:241
-msgid "Picking up deferred messages for wallace"
+msgid "Reproducing verify_recipient(%s, %s) from "
 msgstr ""
 
-#: .././wallace/__init__.py:261
+#: ../bin/kolab_smtp_access_policy.py:756
 #, python-format
-msgid "Picking up deferred messages for module %s"
-msgstr ""
-
-#. Sleep for 300 seconds before reprocessing the deferred queues.
-#. TODO: Consider using queue_run_delay from Postfix, which is where
-#. the default value of 300 seconds comes from.
-#: .././wallace/__init__.py:285
-msgid "Sleeping for 300 seconds"
+msgid "Using authentication domain %s instead of %s"
 msgstr ""
 
-#: .././wallace/__init__.py:310
+#: ../bin/kolab_smtp_access_policy.py:766
 #, python-format
-msgid "Could not bind to socket on port %d on bind "
-msgstr ""
-
-#: .././wallace/__init__.py:320
-msgid "Could not shut down socket"
+msgid "Domain %s is a primary domain"
 msgstr ""
 
-#: .././wallace/__init__.py:394
+#: ../bin/kolab_smtp_access_policy.py:774
 #, python-format
-msgid "Picking up spooled email file %s"
+msgid "Checking the recipient for domain %s that is not "
 msgstr ""
 
-#: .././wallace/__init__.py:424
-msgid "Accepted connection"
+#: ../bin/kolab_smtp_access_policy.py:790
+msgid "This recipient address is related to multiple "
 msgstr ""
 
-#. This is a nested module
-#: .././wallace/modules.py:96
+#: ../bin/kolab_smtp_access_policy.py:799
 #, python-format
-msgid "Module Group: %s"
+msgid "Recipient address %r not found. Allowing since "
 msgstr ""
 
-#: .././wallace/modules.py:107 .././wallace/modules.py:112
-msgid "No such module."
+#: ../bin/kolab_smtp_access_policy.py:827
+msgid "Invalid recipient"
 msgstr ""
 
-#: .././wallace/modules.py:118
-#, python-format
-msgid "Holding message in queue for manual review (%s by %s)"
+#: ../bin/kolab_smtp_access_policy.py:838
+msgid "Could not find this user, accepting"
 msgstr ""
 
-#: .././wallace/modules.py:121
+#: ../bin/kolab_smtp_access_policy.py:901
+#: ../bin/kolab_smtp_access_policy.py:954
 #, python-format
-msgid "Deferring message in %s (by module %s)"
+msgid "Sender %s is not allowed to send to "
 msgstr ""
 
-#: .././wallace/modules.py:131
+#: ../bin/kolab_smtp_access_policy.py:941
 #, python-format
-msgid "The time when the message was sent: %r"
+msgid "Reproducing verify_sender(%r) from cache, "
 msgstr ""
 
-#: .././wallace/modules.py:132
-#, python-format
-msgid "The time now: %r"
+#: ../bin/kolab_smtp_access_policy.py:970
+msgid "Sender uses unauthorized envelope sender address"
 msgstr ""
 
-#: .././wallace/modules.py:133
-#, python-format
-msgid "The time delta: %r"
+#: ../bin/kolab_smtp_access_policy.py:987
+msgid "Could not verify sender"
 msgstr ""
 
-#. TODO: Send NDR back to user
-#: .././wallace/modules.py:137
-#, python-format
-msgid "Message in file %s older then 5 days, deleting"
+#: ../bin/kolab_smtp_access_policy.py:994
+msgid "Verifying whether sender is allowed to send to "
 msgstr ""
 
-#: .././wallace/modules.py:162
+#: ../bin/kolab_smtp_access_policy.py:1008
 #, python-format
-msgid "Rejecting message in %s (by module %s)"
+msgid "Result is %r"
 msgstr ""
 
-#: .././wallace/modules.py:180
-#, python-format
-msgid ""
-"This is the email system Wallace at %s.\n"
-"\n"
-"I'm sorry to inform you we could not deliver the attached message\n"
-"to the following recipients:\n"
-"\n"
-"- %s\n"
-"\n"
-"Your message is being delivered to any other recipients you may have\n"
-"sent your message to. There is no need to resend the message to those\n"
-"recipients.\n"
+#: ../bin/kolab_smtp_access_policy.py:1013
+msgid "No recipient policy restrictions exist for this sender"
 msgstr ""
 
-#: .././wallace/modules.py:195
-#, python-format
-msgid ""
-"X-Wallace-Module: %s\n"
-"X-Wallace-Result: REJECT\n"
+#: ../bin/kolab_smtp_access_policy.py:1022
+msgid "Found a recipient policy to apply for this sender."
 msgstr ""
 
-#: .././wallace/modules.py:248
+#: ../bin/kolab_smtp_access_policy.py:1037
 #, python-format
-msgid "Accepting message in %s (by module %s)"
+msgid "Sender %s not allowed to send to recipient "
 msgstr ""
 
-#: .././wallace/modules.py:316
-#, python-format
-msgid "Module '%s' already registered"
+#: ../bin/kolab_smtp_access_policy.py:1068
+msgid "Cleaning up the cache"
 msgstr ""
 
-#: .././wallace/modules.py:338 .././pykolab/tests/tests.py:183
-#: .././pykolab/cli/commands.py:183 .././pykolab/setup/components.py:243
+#: ../bin/kolab_smtp_access_policy.py:1105
 #, python-format
-msgid "Alias for %s"
+msgid "Operational Error in caching: %s"
 msgstr ""
 
-#: .././wallace/module_optout.py:57
+#: ../bin/kolab_smtp_access_policy.py:1156
 #, python-format
-msgid "Issuing callback after processing to stage %s"
+msgid "Caching the policy result with timestamp %d"
 msgstr ""
 
-#: .././wallace/module_optout.py:58
+#: ../bin/kolab_smtp_access_policy.py:1233
 #, python-format
-msgid "Testing cb_action_%s()"
+msgid "Returning action DEFER_IF_PERMIT: %s"
 msgstr ""
 
-#: .././wallace/module_optout.py:60
+#: ../bin/kolab_smtp_access_policy.py:1238
 #, python-format
-msgid "Attempting to execute cb_action_%s()"
+msgid "Returning action DUNNO: %s"
 msgstr ""
 
-#. modules.next_module('optout')
-#: .././wallace/module_optout.py:66
+#: ../bin/kolab_smtp_access_policy.py:1243
 #, python-format
-msgid "Consulting opt-out service for %r, %r"
+msgid "Returning action HOLD: %s"
 msgstr ""
 
-#: .././wallace/module_optout.py:92
+#: ../bin/kolab_smtp_access_policy.py:1248
 #, python-format
-msgid "Running opt-out consult from envelope sender '%s "
+msgid "Returning action PERMIT: %s"
 msgstr ""
 
-#: .././wallace/module_optout.py:156
+#: ../bin/kolab_smtp_access_policy.py:1253
 #, python-format
-msgid "Attempting to execute cb_action_%s(%r, %r)"
+msgid "Returning action REJECT: %s"
 msgstr ""
 
-#: .././wallace/module_optout.py:177
-#, python-format
-msgid "Could not send request to optout_url %s"
+#: ../bin/kolab_smtp_access_policy.py:1299
+msgid "Starting to loop for new request"
 msgstr ""
 
-#: .././setup-kolab.py:36 .././kolabtest.py:34 .././kolabd.py:31
-#: .././wallace.py:31 .././bonnie.py:31
-msgid "Cannot load pykolab/constants.py:"
+#: ../bin/kolab_smtp_access_policy.py:1306
+msgid "Timeout for policy request reading exceeded"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:89
-msgid "Could not import caching library, caching disabled"
+#: ../bin/kolab_smtp_access_policy.py:1312
+msgid "End of current request"
 msgstr ""
 
-#. Obtain 'kolabDelegate' from the envelope sender.
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:291
+#: ../bin/kolab_smtp_access_policy.py:1316
 #, python-format
-msgid "Obtaining envelope sender dn for %s"
+msgid "Getting line: %s"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:309
-#, python-format
-msgid ""
-"User %s attempted to use envelope sender address %s without authorization"
+#: ../bin/kolab_smtp_access_policy.py:1320
+msgid "Returning request"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:355
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:358
-#, python-format
-msgid "Found user %s to be a valid delegate user of %s"
+#: ../bin/kolab_smtp_access_policy.py:1346
+msgid "Access Policy Options"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:424
-#: .././bin/kolab_smtp_access_policy.py:1301
-msgid "Access Policy Options"
+#: ../bin/kolab_smtp_access_policy.py:1353
+msgid "SMTP Policy request timeout."
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:431
-#: .././bin/kolab_smtp_access_policy.py:1308
+#: ../bin/kolab_smtp_access_policy.py:1359
 msgid "Verify the recipient access policy."
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:437
-#: .././bin/kolab_smtp_access_policy.py:1314
+#: ../bin/kolab_smtp_access_policy.py:1365
 msgid "Verify the sender access policy."
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:443
-#: .././bin/kolab_smtp_access_policy.py:1320
+#: ../bin/kolab_smtp_access_policy.py:1371
 msgid "Allow unauthenticated senders."
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:464
-msgid "Verifying sender."
-msgstr ""
-
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:468
-msgid "No sender specified."
-msgstr ""
-
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:469
-msgid "Invalid sender"
-msgstr ""
-
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:473
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:521
-msgid "No SASL username in request."
-msgstr ""
-
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:475
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:524
-msgid "Not allowing unauthenticated senders."
-msgstr ""
-
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:476
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:525
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:534
-msgid "Access denied"
+#: ../bin/kolab_smtp_access_policy.py:1385
+#, python-format
+msgid "Got request instance %s"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:478
-msgid "Allowing unauthenticated senders."
+#: ../bin/kolab_smtp_access_policy.py:1394
+#, python-format
+msgid "Request instance %s is in state %s"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:484
+#: ../bin/kolab_smtp_access_policy.py:1402
 #, python-format
-msgid "Allowing authenticated sender %s to send as %s."
+msgid "Request instance %s is not yet in DATA state"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:492
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:504
+#: ../bin/kolab_smtp_access_policy.py:1414
 #, python-format
-msgid "Authenticated as sender %s"
+msgid "Request instance %s reached DATA state"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:515
-msgid "Verifying recipient."
+#: ../bin/kolab_smtp_access_policy.py:1432
+msgid "Sender access denied"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:518
-#: .././bin/kolab_smtp_access_policy.py:781
-msgid "Invalid recipient"
+#: ../bin/kolab_smtp_access_policy.py:1434
+msgid "Recipient access denied"
 msgstr ""
 
-#: .././play/kolab-sap/kolab_smtp_access_policy.py:536
-#: .././bin/kolab_smtp_access_policy.py:1383
+#: ../bin/kolab_smtp_access_policy.py:1436
 msgid "No objections"
 msgstr ""
 
-#: .././play/confmgmt/db.py:59 .././play/conf.py:280
-msgid "Dropping all tables..."
-msgstr ""
-
-#: .././play/confmgmt/db.py:62 .././play/conf.py:283
-msgid "Creating the necessary tables..."
-msgstr ""
-
-#: .././play/confmgmt/db.py:69 .././play/conf.py:290
-#: .././play/roundcube_database/__init__.py:57
-#: .././play/roundcube_database/users.py:114 .././pykolab/telemetry.py:652
-msgid "No database available"
-msgstr ""
-
-#: .././play/confmgmt.py:74
-#, python-format
-msgid "Searching for service with name %s"
-msgstr ""
-
-#: .././play/confmgmt.py:77
-#, python-format
-msgid "Found service %s"
+#: ../conf.py:37 ../kolab.py:34 ../saslauthd.py:33
+msgid "Cannot load pykolab/logger.py:"
 msgstr ""
 
-#: .././play/confmgmt.py:82
-#, python-format
-msgid "Searching for role with name %s"
+#: ../kolabd/__init__.py:46 ../saslauthd/__init__.py:47
+#: ../wallace/__init__.py:37
+msgid "Daemon Options"
 msgstr ""
 
-#: .././play/confmgmt.py:85
-#, python-format
-msgid "Found role %s"
+#: ../kolabd/__init__.py:53 ../saslauthd/__init__.py:53
+#: ../wallace/__init__.py:44
+msgid "Fork to the background."
 msgstr ""
 
-#: .././play/confmgmt.py:89
-#, python-format
-msgid "Searching for file with path %s"
+#: ../kolabd/__init__.py:62 ../saslauthd/__init__.py:59
+#: ../wallace/__init__.py:70
+msgid "Path to the PID file to use."
 msgstr ""
 
-#: .././play/confmgmt.py:92
-#, python-format
-msgid "File with path %s not found, adding to database"
+#: ../kolabd/__init__.py:71 ../wallace/__init__.py:87
+msgid "Run as user USERNAME"
 msgstr ""
 
-#: .././play/confmgmt.py:97
-#, python-format
-msgid "Adding file with path %s to service with name %s"
+#: ../kolabd/__init__.py:81 ../wallace/__init__.py:61
+msgid "Run as group GROUPNAME"
 msgstr ""
 
-#: .././play/confmgmt.py:101
+#: ../kolabd/__init__.py:113 ../wallace/__init__.py:448
 #, python-format
-msgid "Adding setting with key %s (%r, %r)"
+msgid "Group %s does not exist"
 msgstr ""
 
-#: .././play/confmgmt.py:106
+#: ../kolabd/__init__.py:122 ../wallace/__init__.py:457
 #, python-format
-msgid "Adding setting %s to file with path %s"
+msgid "Switching real and effective group id to %d"
 msgstr ""
 
-#: .././play/confmgmt.py:110
+#: ../kolabd/__init__.py:144 ../wallace/__init__.py:479
 #, python-format
-msgid "Adding role with name %s to setting with key %s"
+msgid "User %s does not exist"
 msgstr ""
 
-#: .././play/rolequota/__init__.py:53
-#: .././pykolab/plugins/dynamicquota/__init__.py:50
+#: ../kolabd/__init__.py:154 ../wallace/__init__.py:489
 #, python-format
-msgid "No keyword %s passed to set_user_folder_quota"
+msgid "Switching real and effective user id to %d"
 msgstr ""
 
-#: .././play/roundcube_database/__init__.py:51
-#: .././play/roundcube_database/users.py:108
-#, python-format
-msgid "Operational Error in Roundcube database: %s"
+#: ../kolabd/__init__.py:163 ../wallace/__init__.py:498
+msgid "Could not change real and effective uid and/or gid"
 msgstr ""
 
-#: .././kwap/kwap/controllers/secure.py:25
-msgid "Secure Controller here"
+#: ../kolabd/__init__.py:183 ../saslauthd/__init__.py:90
+#: ../wallace/__init__.py:517
+msgid "Interrupted by user"
 msgstr ""
 
-#: .././kwap/kwap/controllers/root.py:103
-msgid "Wrong credentials"
+#: ../kolabd/__init__.py:188 ../kolabd/__init__.py:199
+#: ../wallace/__init__.py:521 ../wallace/__init__.py:531
+msgid "Traceback occurred, please report a "
 msgstr ""
 
-#: .././kwap/kwap/controllers/root.py:118
+#: ../kolabd/__init__.py:194 ../saslauthd/__init__.py:98
+#: ../wallace/__init__.py:527
 #, python-format
-msgid "Welcome back, %s!"
-msgstr ""
-
-#: .././kwap/kwap/controllers/root.py:128
-msgid "We hope to see you soon!"
-msgstr ""
-
-#: .././saslauthd.py:33 .././kolab.py:34 .././conf.py:37
-msgid "Cannot load pykolab/logger.py:"
-msgstr ""
-
-#: .././bonnie/__init__.py:71 .././bonnie/__init__.py:79
-#: .././saslauthd/__init__.py:94 .././saslauthd/__init__.py:102
-msgid "Traceback occurred, please report a bug at http://bugzilla.kolabsys.com"
+msgid "Type Error: %s"
 msgstr ""
 
-#: .././kolabd/__init__.py:116 .././pykolab/auth/ldap/__init__.py:1281
-#: .././pykolab/cli/cmd_sync.py:36
+#: ../kolabd/__init__.py:214 ../pykolab/auth/ldap/__init__.py:1295
+#: ../pykolab/cli/cmd_sync.py:33
 msgid "Listing domains..."
 msgstr ""
 
-#: .././kolabd/__init__.py:148
+#: ../kolabd/__init__.py:246
 #, python-format
 msgid "added domains: %r, removed domains: %r"
 msgstr ""
 
-#: .././pykolab/__init__.py:50
-msgid "Returning thread local configuration"
-msgstr ""
-
-#: .././pykolab/tests/tests.py:53
-msgid "List tests"
-msgstr ""
-
-#. This is a nested test
-#: .././pykolab/tests/tests.py:86
-#, python-format
-msgid "Test Suite: %s"
-msgstr ""
-
-#: .././pykolab/tests/tests.py:100 .././pykolab/tests/tests.py:105
-msgid "No such test."
-msgstr ""
-
-#: .././pykolab/tests/tests.py:157 .././pykolab/tests/tests.py:161
-#, python-format
-msgid "Test '%s' already registered"
-msgstr ""
-
-#: .././pykolab/tests/tests.py:191 .././pykolab/cli/commands.py:191
-#: .././pykolab/setup/components.py:251
-msgid "Not yet implemented"
-msgstr ""
-
-#: .././pykolab/tests/__init__.py:39
-msgid "Test Options"
-msgstr ""
-
-#: .././pykolab/tests/__init__.py:45
-msgid "Run tests in suite SUITE. Implies a certain set of items being tested."
-msgstr ""
-
-#: .././pykolab/tests/imap/test_login.py:44
-#, python-format
-msgid "Connecting at %s"
-msgstr ""
-
-#: .././pykolab/tests/imap/test_login.py:46
-#, python-format
-msgid "Connected at %s"
+#: ../kolabd.py:31 ../kolabtest.py:34 ../setup-kolab.py:36 ../wallace.py:31
+msgid "Cannot load pykolab/constants.py:"
 msgstr ""
 
-#: .././pykolab/tests/imap/test_login.py:51
+#: ../pykolab/auth/__init__.py:94
 #, python-format
-msgid "Logging in at %s"
+msgid "Called for domain %r"
 msgstr ""
 
-#: .././pykolab/tests/imap/test_login.py:53
+#: ../pykolab/auth/__init__.py:110
 #, python-format
-msgid "Logged in at %s"
+msgid "Connecting to Authentication backend for domain %s"
 msgstr ""
 
-#: .././pykolab/tests/imap/test_login.py:60
+#: ../pykolab/auth/ldap/cache.py:109
 #, python-format
-msgid "Test failure in %s"
+msgid "Inserting cache entry %r"
 msgstr ""
 
-#: .././pykolab/auth/__init__.py:94
+#: ../pykolab/auth/ldap/cache.py:122
 #, python-format
-msgid "Called for domain %r"
+msgid "Updating timestamp for cache entry %r"
 msgstr ""
 
-#: .././pykolab/auth/__init__.py:110
+#: ../pykolab/auth/ldap/cache.py:129
 #, python-format
-msgid "Connecting to Authentication backend for domain %s"
+msgid "Updating result_attribute for cache entry %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:51
+#: ../pykolab/auth/ldap/__init__.py:51
 msgid "Python LDAP library does not support persistent search"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:141
+#: ../pykolab/auth/ldap/__init__.py:141
 #, python-format
 msgid "Attempting to authenticate user %s in realm %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:181
+#: ../pykolab/auth/ldap/__init__.py:181
 #, python-format
 msgid "Binding with user_dn %s and password %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:190
+#: ../pykolab/auth/ldap/__init__.py:190
 #, python-format
 msgid "Failed to authenticate as user %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:205
+#: ../pykolab/auth/ldap/__init__.py:205
 msgid "Connecting to LDAP..."
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:209
+#: ../pykolab/auth/ldap/__init__.py:209
 #, python-format
 msgid "Attempting to use LDAP URI %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:345
+#: ../pykolab/auth/ldap/__init__.py:349
 #, python-format
 msgid "Finding recipient with filter %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:390
+#: ../pykolab/auth/ldap/__init__.py:394
 #, python-format
 msgid "Applying recipient policy to %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:409
+#: ../pykolab/auth/ldap/__init__.py:413
 #, python-format
 msgid "Using mail attributes: %r, with primary %r and "
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:420
+#: ../pykolab/auth/ldap/__init__.py:424
 #, python-format
 msgid "key %r not in entry"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:422
+#: ../pykolab/auth/ldap/__init__.py:426
 #, python-format
 msgid "key %r is the prim. mail attr."
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:424
+#: ../pykolab/auth/ldap/__init__.py:428
 msgid "prim. mail pol. is not empty"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:427
+#: ../pykolab/auth/ldap/__init__.py:431
 #, python-format
 msgid "key %r is the sec. mail attr."
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:429
+#: ../pykolab/auth/ldap/__init__.py:433
 msgid "sec. mail pol. is not empty"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:432
-#: .././pykolab/auth/ldap/__init__.py:445
+#: ../pykolab/auth/ldap/__init__.py:436 ../pykolab/auth/ldap/__init__.py:449
 #, python-format
 msgid "Attributes %r are not yet available for entry %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:483
+#: ../pykolab/auth/ldap/__init__.py:487
 #, python-format
 msgid "No results for mail address %s found"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:494
+#: ../pykolab/auth/ldap/__init__.py:498
 #, python-format
 msgid "1 result for address %s found, verifying"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:504
+#: ../pykolab/auth/ldap/__init__.py:508
 #, python-format
 msgid "Too bad, primary email address %s "
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:515
-#: .././pykolab/auth/ldap/__init__.py:604
+#: ../pykolab/auth/ldap/__init__.py:519 ../pykolab/auth/ldap/__init__.py:608
 msgid "Address assigned to us"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:570
+#: ../pykolab/auth/ldap/__init__.py:574
 #, python-format
 msgid "No results for address %s found"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:581
+#: ../pykolab/auth/ldap/__init__.py:585
 #, python-format
 msgid "1 result for address %s found, "
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:592
+#: ../pykolab/auth/ldap/__init__.py:596
 msgid "Too bad, secondary email "
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:646
+#: ../pykolab/auth/ldap/__init__.py:650
 #, python-format
 msgid "Entry modifications list: %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:688
+#: ../pykolab/auth/ldap/__init__.py:693
 #, python-format
 msgid "Using filter %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:726
+#: ../pykolab/auth/ldap/__init__.py:731
 msgid "Invalid bind credentials"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:979
+#: ../pykolab/auth/ldap/__init__.py:984
 #, python-format
 msgid "Current changelog entry %s with %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:986
+#: ../pykolab/auth/ldap/__init__.py:991
 #, python-format
 msgid "Result from recipient policy: %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1217
+#: ../pykolab/auth/ldap/__init__.py:1095
+#, python-format
+msgid "Kolab user %s does not have a result attribute %r"
+msgstr ""
+
+#: ../pykolab/auth/ldap/__init__.py:1231
 #, python-format
 msgid "Finding domain root dn for domain %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1305
+#: ../pykolab/auth/ldap/__init__.py:1319
 msgid "Authentication database DOWN"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1366
+#: ../pykolab/auth/ldap/__init__.py:1380
 #, python-format
 msgid "Entry type: %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1439
+#: ../pykolab/auth/ldap/__init__.py:1453
 msgid "LDAP Search Result Data Entry:"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1455
+#: ../pykolab/auth/ldap/__init__.py:1469
 msgid "Entry Change Notification attributes:"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1460
+#: ../pykolab/auth/ldap/__init__.py:1474
 #, python-format
 msgid "Change Type: %r (%r)"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1468
+#: ../pykolab/auth/ldap/__init__.py:1482
 #, python-format
 msgid "Previous DN: %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1522
+#: ../pykolab/auth/ldap/__init__.py:1536
 #, python-format
 msgid "Object %s searched no longer exists"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1534
+#: ../pykolab/auth/ldap/__init__.py:1548
 #, python-format
 msgid "%d results..."
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1624
+#: ../pykolab/auth/ldap/__init__.py:1638
 #, python-format
 msgid "Searching with filter %r"
 msgstr ""
 
-#: .././pykolab/auth/ldap/__init__.py:1668
+#: ../pykolab/auth/ldap/__init__.py:1682
 #, python-format
 msgid "Checking for support for %s on %s"
 msgstr ""
 
-#: .././pykolab/auth/ldap/cache.py:109
-#, python-format
-msgid "Inserting cache entry %r"
+#: ../pykolab/cli/cmd_add_domain.py:36 ../pykolab/cli/cmd_create_mailbox.py:36
+#: ../pykolab/cli/cmd_export_mailbox.py:33
+#: ../pykolab/cli/cmd_list_mailboxes.py:39
+msgid "CLI Options"
 msgstr ""
 
-#: .././pykolab/auth/ldap/cache.py:122
-#, python-format
-msgid "Updating timestamp for cache entry %r"
+#: ../pykolab/cli/cmd_add_domain.py:42
+msgid "Add domain as alias for DOMAIN"
 msgstr ""
 
-#: .././pykolab/auth/ldap/cache.py:128
-#, python-format
-msgid "Updating result_attribute for cache entry %r"
+#: ../pykolab/cli/cmd_add_domain.py:47
+msgid "Add a new domain or domain alias."
 msgstr ""
 
-#: .././pykolab/auth/sql/__init__.py:44
-msgid "Connecting to SQL..."
+#: ../pykolab/cli/cmd_add_domain.py:55
+msgid "Could not find credentials with sufficient permissions"
 msgstr ""
 
-#: .././pykolab/utils.py:92
-msgid "Please answer 'yes' or 'no'."
+#: ../pykolab/cli/cmd_add_domain.py:80
+msgid "Invalid parent domain"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:87
-#, python-format
-msgid "Setting %s to %r (from defaults)"
+#: ../pykolab/cli/cmd_add_domain.py:86
+msgid "Domain name"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:106
-#, python-format
-msgid "Setting %s to %r (from CLI, verified)"
+#: ../pykolab/cli/cmd_create_mailbox.py:41
+msgid "Set metadata for folder to ANNOTATION=VALUE"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:109
-#, python-format
-msgid "Setting %s to %r (from CLI, not checked)"
+#: ../pykolab/cli/cmd_create_mailbox.py:50
+msgid "Invalid argument"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:150 .././pykolab/conf/__init__.py:207
-#, python-format
-msgid "Setting %s_%s to '****' (from configuration file)"
+#: ../pykolab/cli/cmd_create_mailbox.py:58
+msgid "Invalid argument for metadata"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:152 .././pykolab/conf/__init__.py:209
-#, python-format
-msgid "Setting %s_%s to %r (from configuration file)"
+#: ../pykolab/cli/cmd_delete_mailbox.py:46
+msgid "No mailbox specified"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:162
-msgid "Setting options from configuration file"
+#: ../pykolab/cli/cmd_export_mailbox.py:38
+msgid "All folders this user has access to"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:223
+#: ../pykolab/cli/cmd_export_mailbox.py:108
 #, python-format
-msgid "Configuration file %s not readable"
+msgid "%s is not a directory"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:226
+#: ../pykolab/cli/cmd_export_mailbox.py:118
 #, python-format
-msgid "Reading configuration file %s"
+msgid "ZIP file at %s.zip"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:230
+#: ../pykolab/cli/cmd_export_mailbox.py:120
 #, python-format
-msgid "Invalid configuration file %s"
+msgid "No directories found for user %s"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:233
+#: ../pykolab/cli/cmd_list_mailboxes.py:44
+msgid "Display raw UTF-7 folder names"
+msgstr ""
+
+#: ../pykolab/cli/cmd_list_mailboxes.py:75
 #, python-format
-msgid "No master configuration section [kolab] in configuration file %s"
+msgid "Appending folder search for %r"
+msgstr ""
+
+#: ../pykolab/cli/cmd_remove_mailaddress.py:49
+msgid "Invalid or unqualified email address."
+msgstr ""
+
+#: ../pykolab/cli/cmd_remove_mailaddress.py:57
+#, python-format
+msgid "No recipient found for email address %r"
+msgstr ""
+
+#: ../pykolab/cli/cmd_remove_mailaddress.py:60
+#, python-format
+msgid "Found the following recipient(s): %r"
+msgstr ""
+
+#: ../pykolab/cli/cmd_remove_mailaddress.py:66
+#, python-format
+msgid "Using the following mail attributes: %r"
+msgstr ""
+
+#: ../pykolab/cli/cmd_remove_mailaddress.py:90
+msgid "Found the following recipients:"
+msgstr ""
+
+#: ../pykolab/cli/cmd_sync.py:38
+#, python-format
+msgid "Found %d domains in %d seconds"
+msgstr ""
+
+#: ../pykolab/cli/cmd_sync.py:48
+#, python-format
+msgid "Running for domain %s"
+msgstr ""
+
+#: ../pykolab/cli/cmd_sync.py:54
+#, python-format
+msgid "Synchronizing users for %s took %d seconds"
+msgstr ""
+
+#. This is a nested command
+#. This is a nested component
+#: ../pykolab/cli/commands.py:98 ../pykolab/setup/components.py:88
+#, python-format
+msgid "Command Group: %s"
+msgstr ""
+
+#: ../pykolab/cli/commands.py:109 ../pykolab/cli/commands.py:114
+msgid "No such command."
+msgstr ""
+
+#: ../pykolab/cli/commands.py:165 ../pykolab/setup/components.py:217
+#, python-format
+msgid "Command '%s' already registered"
+msgstr ""
+
+#: ../pykolab/cli/commands.py:190 ../pykolab/setup/components.py:243
+#: ../pykolab/tests/tests.py:183 ../wallace/modules.py:338
+#, python-format
+msgid "Alias for %s"
+msgstr ""
+
+#: ../pykolab/cli/commands.py:198 ../pykolab/setup/components.py:251
+#: ../pykolab/tests/tests.py:191
+msgid "Not yet implemented"
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_examine_command_issue.py:40
+msgid "Unspecified command issue identifier"
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_examine_command_issue.py:50
+msgid "Invalid command issue identifier"
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_examine_command_issue.py:60
+#: ../pykolab/cli/telemetry/cmd_examine_session.py:65
+msgid "Invalid session identifier"
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_examine_command_issue.py:75
+#: ../pykolab/cli/telemetry/cmd_examine_session.py:100
+#, python-format
+msgid "Session by %s on server %s"
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_examine_session.py:43
+msgid "Unspecified session identifier"
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_list_sessions.py:35
+msgid "List Options"
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_list_sessions.py:40
+msgid "Display sessions since ..."
+msgstr ""
+
+#: ../pykolab/cli/telemetry/cmd_list_sessions.py:58
+#, python-format
+msgid "Session for user %s started at %s with ID %s"
+msgstr ""
+
+#: ../pykolab/conf/entitlement.py:57 ../pykolab/conf/entitlement.py:80
+#: ../pykolab/conf/entitlement.py:98 ../pykolab/conf/entitlement.py:116
+#: ../pykolab/conf/entitlement.py:229 ../pykolab/conf/entitlement.py:245
+#: ../pykolab/conf/entitlement.py:261
+msgid "Invalid entitlement verification "
+msgstr ""
+
+#: ../pykolab/conf/entitlement.py:120 ../pykolab/conf/entitlement.py:123
+msgid "Invalid entitlement certificate"
+msgstr ""
+
+#: ../pykolab/conf/entitlement.py:132
+#, python-format
+msgid "Parsing entitlement file %s"
+msgstr ""
+
+#: ../pykolab/conf/entitlement.py:141
+#, python-format
+msgid "License file %s not readable!"
+msgstr ""
+
+#: ../pykolab/conf/entitlement.py:146
+msgid "No entitlement directory found"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:87
+#, python-format
+msgid "Setting %s to %r (from defaults)"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:106
+#, python-format
+msgid "Setting %s to %r (from CLI, verified)"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:109
+#, python-format
+msgid "Setting %s to %r (from CLI, not checked)"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:150 ../pykolab/conf/__init__.py:207
+#, python-format
+msgid "Setting %s_%s to '****' (from configuration file)"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:152 ../pykolab/conf/__init__.py:209
+#, python-format
+msgid "Setting %s_%s to %r (from configuration file)"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:162
+msgid "Setting options from configuration file"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:223
+#, python-format
+msgid "Configuration file %s not readable"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:226
+#, python-format
+msgid "Reading configuration file %s"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:230
+#, python-format
+msgid "Invalid configuration file %s"
+msgstr ""
+
+#: ../pykolab/conf/__init__.py:233
+#, python-format
+msgid "No master configuration section [kolab] in configuration file %s"
 msgstr ""
 
 #. #
 #. # Runtime Options
 #. #
-#: .././pykolab/conf/__init__.py:267
+#: ../pykolab/conf/__init__.py:267
 msgid "Runtime Options"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:272
+#: ../pykolab/conf/__init__.py:272
 msgid "Configuration file to use"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:278
+#: ../pykolab/conf/__init__.py:278
 msgid "Set the debugging "
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:286
+#: ../pykolab/conf/__init__.py:286
 msgid "Set the logging level. "
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:294
+#: ../pykolab/conf/__init__.py:294
 msgid "Log file to use"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:300
+#: ../pykolab/conf/__init__.py:300
 msgid "Be quiet."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:306
+#: ../pykolab/conf/__init__.py:306
 msgid "Answer yes to all questions."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:328
+#: ../pykolab/conf/__init__.py:328
 msgid "No command supplied"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:411
+#: ../pykolab/conf/__init__.py:411
 msgid "Insufficient options. Need section, key and value -in that order."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:414
+#: ../pykolab/conf/__init__.py:414
 #, python-format
 msgid "No section '%s' exists."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:445
+#: ../pykolab/conf/__init__.py:445
 #, python-format
 msgid "Setting %s to %r (from the default values for CLI options)"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:514
+#: ../pykolab/conf/__init__.py:514
 #, python-format
 msgid "Could not execute configuration function: %s"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:522
+#: ../pykolab/conf/__init__.py:522
 #, python-format
 msgid "Option %s/%s does not exist in config file %s, pulling from defaults"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:530 .././pykolab/conf/__init__.py:533
+#: ../pykolab/conf/__init__.py:530 ../pykolab/conf/__init__.py:533
 msgid "Option does not exist in defaults."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:531 .././pykolab/conf/__init__.py:534
+#: ../pykolab/conf/__init__.py:531 ../pykolab/conf/__init__.py:534
 msgid "Not available"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:543
+#: ../pykolab/conf/__init__.py:543
 #, python-format
 msgid "Configuration file %s not readable."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:546
+#: ../pykolab/conf/__init__.py:546
 #, python-format
 msgid "Configuration file %s does not exist."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:551
+#: ../pykolab/conf/__init__.py:551
 msgid ""
 "WARNING: A negative debug level value does not make this program be any more "
 "silent."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:557
+#: ../pykolab/conf/__init__.py:557
 msgid "This program has 9 levels of verbosity. Using the maximum of 9."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:565 .././pykolab/conf/__init__.py:571
+#: ../pykolab/conf/__init__.py:565 ../pykolab/conf/__init__.py:571
 msgid "Cannot start SASL authentication daemon"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:582
+#: ../pykolab/conf/__init__.py:582
 msgid "No imaplib library found."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:592
+#: ../pykolab/conf/__init__.py:592
 msgid "No LMTP class found in the smtplib library."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:602
+#: ../pykolab/conf/__init__.py:602
 msgid "No SMTP class found in the smtplib library."
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:616
+#: ../pykolab/conf/__init__.py:616
 #, python-format
 msgid "Found you specified a specific set of items to test: %s"
 msgstr ""
 
-#: .././pykolab/conf/__init__.py:624
+#: ../pykolab/conf/__init__.py:624
 #, python-format
 msgid "Selectively selecting: %s"
 msgstr ""
 
-#: .././pykolab/conf/entitlement.py:57 .././pykolab/conf/entitlement.py:80
-#: .././pykolab/conf/entitlement.py:98 .././pykolab/conf/entitlement.py:116
-#: .././pykolab/conf/entitlement.py:229 .././pykolab/conf/entitlement.py:245
-#: .././pykolab/conf/entitlement.py:261
-msgid "Invalid entitlement verification "
-msgstr ""
-
-#: .././pykolab/conf/entitlement.py:120 .././pykolab/conf/entitlement.py:123
-msgid "Invalid entitlement certificate"
-msgstr ""
-
-#: .././pykolab/conf/entitlement.py:132
-#, python-format
-msgid "Parsing entitlement file %s"
-msgstr ""
-
-#: .././pykolab/conf/entitlement.py:141
-#, python-format
-msgid "License file %s not readable!"
-msgstr ""
-
-#: .././pykolab/conf/entitlement.py:146
-msgid "No entitlement directory found"
-msgstr ""
-
-#: .././pykolab/logger.py:89
-#, python-format
-msgid "Cannot log to file %s: %s"
-msgstr ""
-
-#: .././pykolab/constants.py:40
+#: ../pykolab/constants.py.in:40
 msgid "PyKolab is a Kolab Systems product. For more information "
 msgstr ""
 
-#: .././pykolab/constants.py:54
+#: ../pykolab/constants.py.in:54
 msgid "WARNING"
 msgstr ""
 
-#: .././pykolab/constants.py:54
+#: ../pykolab/constants.py.in:54
 msgid "The Fully Qualified "
 msgstr ""
 
-#: .././pykolab/constants.py:75
+#: ../pykolab/constants.py.in:75
 msgid "389 Directory Server or Red Hat Directory Server"
 msgstr ""
 
-#: .././pykolab/constants.py:79 .././pykolab/constants.py:83
+#: ../pykolab/constants.py.in:79 ../pykolab/constants.py.in:83
 msgid "OpenLDAP or compatible"
 msgstr ""
 
-#: .././pykolab/cli/telemetry/cmd_list_sessions.py:38
-msgid "List Options"
-msgstr ""
-
-#: .././pykolab/cli/telemetry/cmd_list_sessions.py:43
-msgid "Display sessions since ..."
+#: ../pykolab/imap/cyrus.py:78
+#, python-format
+msgid "Could not connect to Cyrus IMAP server %r"
 msgstr ""
 
-#: .././pykolab/cli/telemetry/cmd_list_sessions.py:61
+#: ../pykolab/imap/cyrus.py:101
 #, python-format
-msgid "Session for user %s started at %s with ID %s"
+msgid "Continuing with separator: %r"
 msgstr ""
 
-#: .././pykolab/cli/telemetry/cmd_examine_session.py:46
-msgid "Unspecified session identifier"
+#: ../pykolab/imap/cyrus.py:106
+msgid "Detected we are running in a Murder topology"
 msgstr ""
 
-#: .././pykolab/cli/telemetry/cmd_examine_session.py:68
-#: .././pykolab/cli/telemetry/cmd_examine_command_issue.py:63
-msgid "Invalid session identifier"
+#: ../pykolab/imap/cyrus.py:110
+msgid "This system is not part of a murder topology"
 msgstr ""
 
-#: .././pykolab/cli/telemetry/cmd_examine_session.py:103
-#: .././pykolab/cli/telemetry/cmd_examine_command_issue.py:78
+#: ../pykolab/imap/cyrus.py:131
 #, python-format
-msgid "Session by %s on server %s"
+msgid "Checking actual backend server for folder %s through annotations"
 msgstr ""
 
-#: .././pykolab/cli/telemetry/cmd_examine_command_issue.py:43
-msgid "Unspecified command issue identifier"
+#: ../pykolab/imap/cyrus.py:146
+#, python-format
+msgid "Could not get the annotations after %s tries."
 msgstr ""
 
-#: .././pykolab/cli/telemetry/cmd_examine_command_issue.py:53
-msgid "Invalid command issue identifier"
+#: ../pykolab/imap/cyrus.py:150
+#, python-format
+msgid "No annotations for %s: %r"
 msgstr ""
 
-#. This is a nested command
-#. This is a nested component
-#: .././pykolab/cli/commands.py:95 .././pykolab/setup/components.py:88
+#: ../pykolab/imap/cyrus.py:157
 #, python-format
-msgid "Command Group: %s"
+msgid "Server for INBOX folder %s is %s"
 msgstr ""
 
-#: .././pykolab/cli/commands.py:106 .././pykolab/cli/commands.py:111
-msgid "No such command."
+#: ../pykolab/imap/cyrus.py:169
+#, python-format
+msgid "Setting quota for INBOX folder %s to %s"
 msgstr ""
 
-#: .././pykolab/cli/commands.py:161 .././pykolab/setup/components.py:217
+#: ../pykolab/imap/cyrus.py:173
 #, python-format
-msgid "Command '%s' already registered"
+msgid "Could not set quota for mailfolder %s"
 msgstr ""
 
-#: .././pykolab/cli/cmd_delete_mailbox.py:46
-msgid "No mailbox specified"
+#: ../pykolab/imap/cyrus.py:182
+#, python-format
+msgid "Moving INBOX folder %s to %s"
 msgstr ""
 
-#: .././pykolab/cli/cmd_list_mailboxes.py:38
-#: .././pykolab/cli/cmd_export_mailbox.py:36
-msgid "CLI Options"
+#: ../pykolab/imap/cyrus.py:194
+#, python-format
+msgid "Setting annotation %s on folder %s"
 msgstr ""
 
-#: .././pykolab/cli/cmd_list_mailboxes.py:43
-msgid "Display raw UTF-7 folder names"
+#: ../pykolab/imap/cyrus.py:201
+#, python-format
+msgid "Could not set annotation %r on mail folder %r: %r"
 msgstr ""
 
-#: .././pykolab/cli/cmd_export_mailbox.py:41
-msgid "All folders this user has access to"
+#: ../pykolab/imap/cyrus.py:205
+#, python-format
+msgid "Transferring folder %s from %s to %s"
 msgstr ""
 
-#: .././pykolab/cli/cmd_export_mailbox.py:111
+#: ../pykolab/imap/cyrus.py:265
 #, python-format
-msgid "%s is not a directory"
+msgid "Undeleting %s to %s"
 msgstr ""
 
-#: .././pykolab/cli/cmd_export_mailbox.py:121
+#: ../pykolab/imap/__init__.py:45
 #, python-format
-msgid "ZIP file at %s.zip"
+msgid "Cleaning up ACL entries for %s across all folders"
 msgstr ""
 
-#: .././pykolab/cli/cmd_export_mailbox.py:123
+#: ../pykolab/imap/__init__.py:60
 #, python-format
-msgid "No directories found for user %s"
+msgid "Cleaning up ACL entries referring to identifier %s"
 msgstr ""
 
-#: .././pykolab/cli/cmd_sync.py:41
+#: ../pykolab/imap/__init__.py:69
 #, python-format
-msgid "Found %d domains in %d seconds"
+msgid "Iterating over %d folders"
 msgstr ""
 
-#: .././pykolab/cli/cmd_sync.py:51
+#. Set the ACL to '' (effectively deleting the ACL entry)
+#: ../pykolab/imap/__init__.py:82
 #, python-format
-msgid "Running for domain %s"
+msgid "Removing acl %r for subject %r from folder %r"
 msgstr ""
 
-#: .././pykolab/cli/cmd_sync.py:57
+#: ../pykolab/imap/__init__.py:143
 #, python-format
-msgid "Synchronizing users for %s took %d seconds"
+msgid "Logging on to Cyrus IMAP server %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:70
+#: ../pykolab/imap/__init__.py:152
 #, python-format
-msgid "ImportError for plugin %s: %s"
+msgid "Logging on to Dovecot IMAP server %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:74
+#: ../pykolab/imap/__init__.py:161
 #, python-format
-msgid "RuntimeError for plugin %s: %s"
+msgid "Logging on to generic IMAP server %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:78
+#: ../pykolab/imap/__init__.py:179
 #, python-format
-msgid "Plugin %s failed to load (%s: %s)"
+msgid "Reusing existing IMAP server connection to %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:116 .././pykolab/plugins/__init__.py:118
+#: ../pykolab/imap/__init__.py:181
 #, python-format
-msgid "Cannot set defaults for plugin %s: %s"
+msgid "Reconnecting to IMAP server %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:120
-#, python-format
-msgid "Cannot set defaults for plugin %s: Unknown Error"
+#: ../pykolab/imap/__init__.py:197
+msgid "Called imap.disconnect() on a server that "
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:123
+#: ../pykolab/imap/__init__.py:213
 #, python-format
-msgid "Not setting defaults for plugin %s: No function 'set_defaults()'"
+msgid "%r has no attribute %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:142
+#: ../pykolab/imap/__init__.py:221 ../pykolab/imap/__init__.py:239
 #, python-format
-msgid "Cannot set runtime for plugin %s: %s"
+msgid "Creating new shared folder %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:144
+#: ../pykolab/imap/__init__.py:255
 #, python-format
-msgid "Not setting runtime for plugin %s: No function 'set_runtime()'"
+msgid "Creating new mailbox for user %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:163 .././pykolab/plugins/__init__.py:165
+#: ../pykolab/imap/__init__.py:280
 #, python-format
-msgid "Cannot add options for plugin %s: %s"
+msgid "Creating additional folders for user %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:167
+#: ../pykolab/imap/__init__.py:302
 #, python-format
-msgid "Not adding options for plugin %s: No function 'add_options()'"
+msgid "Mailbox already exists: %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:187
-#, python-format
-msgid "Cannot check options for plugin %s: %s"
+#: ../pykolab/imap/__init__.py:345
+msgid "Subscribing user to the additional folders"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:189
+#: ../pykolab/imap/__init__.py:380 ../pykolab/imap/__init__.py:459
 #, python-format
-msgid "Not checking options for plugin %s: No function 'check_options()'"
+msgid "Renaming INBOX from %s to %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:227
+#: ../pykolab/imap/__init__.py:384
 #, python-format
-msgid "Executing hook %s for plugin %s"
+msgid "Could not rename INBOX folder %s to %s"
 msgstr ""
 
-#: .././pykolab/plugins/__init__.py:231 .././pykolab/plugins/__init__.py:233
+#: ../pykolab/imap/__init__.py:386 ../pykolab/imap/__init__.py:463
 #, python-format
-msgid "Cannot execute hook %s for plugin %s: %s"
+msgid "Moving INBOX folder %s won't succeed as target folder %s already exists"
 msgstr ""
 
-#: .././pykolab/plugins/defaultfolders/__init__.py:47
+#: ../pykolab/imap/__init__.py:399
 #, python-format
-msgid "Plugin %s called without required keyword %s."
+msgid "Looking for folder '%s', we found folders: %r"
 msgstr ""
 
-#: .././pykolab/plugins/defaultfolders/__init__.py:53
-msgid "Could not parse additional_folders"
+#: ../pykolab/imap/__init__.py:422
+#, python-format
+msgid "Setting ACL rights %s for subject %s on folder "
 msgstr ""
 
-#: .././pykolab/plugins/dynamicquota/__init__.py:56
+#: ../pykolab/imap/__init__.py:433
 #, python-format
-msgid "Quota '%s' not an integer!"
+msgid "Removing ACL rights %s for subject %s on folder "
 msgstr ""
 
-#: .././pykolab/plugins/dynamicquota/__init__.py:63
-msgid ""
-"The new quota was set to 0, but default quota > 0, returning default quota"
+#: ../pykolab/imap/__init__.py:456
+#, python-format
+msgid "Found old INBOX folder %s"
 msgstr ""
 
-#: .././pykolab/plugins/dynamicquota/__init__.py:82
-msgid ""
-"The default quota is larger then the calculated new quota, using the default "
-"quota"
+#: ../pykolab/imap/__init__.py:465
+#, python-format
+msgid "Did not find old folder user/%s to rename"
 msgstr ""
 
-#: .././pykolab/plugins/recipientpolicy/__init__.py:70
-msgid "Attribute substitution for 'mail' failed in Recipient Policy"
+#: ../pykolab/imap/__init__.py:467
+msgid "Value for user is not a dictionary"
 msgstr ""
 
-#: .././pykolab/plugins/recipientpolicy/__init__.py:97
-msgid "Could not parse the alternative mail routines"
+#. TODO: Go in fact correct the quota.
+#: ../pykolab/imap/__init__.py:527
+#, python-format
+msgid "Cannot get current IMAP quota for folder %s"
 msgstr ""
 
-#: .././pykolab/plugins/recipientpolicy/__init__.py:109
-#: .././pykolab/plugins/recipientpolicy/__init__.py:119
-msgid ""
-"Attribute substitution for 'alternative_mail' failed in Recipient Policy"
+#: ../pykolab/imap/__init__.py:540
+#, python-format
+msgid "Quota for %s currently is %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_zpush.py:40
-msgid "Setup zpush."
+#: ../pykolab/imap/__init__.py:546
+#, python-format
+msgid "Adjusting authentication database quota for folder %s to %d"
 msgstr ""
 
-#: .././pykolab/setup/setup_zpush.py:44
-msgid "Z-Push is not installed on this system"
+#: ../pykolab/imap/__init__.py:551
+#, python-format
+msgid "Correcting quota for %s to %s (currently %s)"
 msgstr ""
 
-#: .././pykolab/setup/setup_zpush.py:70
-#: .././pykolab/setup/setup_roundcube.py:93
-#: .././pykolab/setup/setup_freebusy.py:76
+#: ../pykolab/imap/__init__.py:633
 #, python-format
-msgid "Using template file %r"
+msgid "Checking folder: %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_zpush.py:77
-#: .././pykolab/setup/setup_roundcube.py:100
-#: .././pykolab/setup/setup_freebusy.py:83
+#: ../pykolab/imap/__init__.py:638
 #, python-format
-msgid "Successfully compiled template %r, writing out to %r"
+msgid "Folder has no corresponding user (1): %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_zpush.py:95 .././pykolab/setup/setup_ldap.py:184
-#: .././pykolab/setup/setup_ldap.py:358
-#: .././pykolab/setup/setup_roundcube.py:159
-#: .././pykolab/setup/setup_imap.py:137 .././pykolab/setup/setup_mysql.py:49
-#: .././pykolab/setup/setup_freebusy.py:101
-msgid "Could not start and configure to start on boot, the "
+#: ../pykolab/imap/__init__.py:641
+#, python-format
+msgid "Folder has no corresponding user (2): %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:43
-msgid "LDAP Options"
+#. We got user identifier only
+#: ../pykolab/imap/__init__.py:656
+msgid "Please don't give us just a user identifier"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:50
-msgid "Specify FQDN (overriding defaults)."
+#: ../pykolab/imap/__init__.py:659
+#, python-format
+msgid "Deleting folder %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:54
-msgid "Setup LDAP."
+#: ../pykolab/__init__.py:50
+msgid "Returning thread local configuration"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:66
-msgid "Administrator password"
+#: ../pykolab/logger.py:105
+#, python-format
+msgid "Could not change the ownership of log file %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:72
-msgid "Directory Manager password"
+#: ../pykolab/logger.py:121
+#, python-format
+msgid "Cannot log to file %s: %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:77
-msgid "User"
+#: ../pykolab/plugins/defaultfolders/__init__.py:47
+#, python-format
+msgid "Plugin %s called without required keyword %s."
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:78
-msgid "Group"
+#: ../pykolab/plugins/defaultfolders/__init__.py:53
+msgid "Could not parse additional_folders"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:143
-msgid "Setting up 389 Directory Server"
+#: ../pykolab/plugins/dynamicquota/__init__.py:50
+#, python-format
+msgid "No keyword %s passed to set_user_folder_quota"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:153
-msgid "Setup DS stdout:"
+#: ../pykolab/plugins/dynamicquota/__init__.py:56
+#, python-format
+msgid "Quota '%s' not an integer!"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:156
-msgid "Setup DS stderr:"
+#: ../pykolab/plugins/dynamicquota/__init__.py:63
+msgid ""
+"The new quota was set to 0, but default quota > 0, returning default quota"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:175 .././pykolab/setup/setup_mysql.py:97
-msgid "Could not find the Kolab schema file"
+#: ../pykolab/plugins/dynamicquota/__init__.py:82
+msgid ""
+"The default quota is larger then the calculated new quota, using the default "
+"quota"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:189
-msgid "Cyrus Administrator password"
+#: ../pykolab/plugins/__init__.py:70
+#, python-format
+msgid "ImportError for plugin %s: %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:195
-msgid "Kolab Service password"
+#: ../pykolab/plugins/__init__.py:74
+#, python-format
+msgid "RuntimeError for plugin %s: %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:204
-msgid "Writing out configuration to kolab.conf"
+#: ../pykolab/plugins/__init__.py:78
+#, python-format
+msgid "Plugin %s failed to load (%s: %s)"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:218
-msgid "Inserting service users into LDAP."
+#: ../pykolab/plugins/__init__.py:116 ../pykolab/plugins/__init__.py:118
+#, python-format
+msgid "Cannot set defaults for plugin %s: %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:262
-msgid "Writing out cn=kolab,cn=config"
+#: ../pykolab/plugins/__init__.py:120
+#, python-format
+msgid "Cannot set defaults for plugin %s: Unknown Error"
 msgstr ""
 
-#. TODO: Add kolab-admin role
-#. TODO: Assign kolab-admin admin ACLs
-#: .././pykolab/setup/setup_ldap.py:286
+#: ../pykolab/plugins/__init__.py:123
 #, python-format
-msgid "Adding domain %s to list of domains for this deployment"
+msgid "Not setting defaults for plugin %s: No function 'set_defaults()'"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:294
-msgid "Disabling anonymous binds"
+#: ../pykolab/plugins/__init__.py:142
+#, python-format
+msgid "Cannot set runtime for plugin %s: %s"
 msgstr ""
 
-#. TODO: Ensure the uid attribute is unique
-#. TODO^2: Consider renaming the general "attribute uniqueness to "uid attribute uniqueness"
-#: .././pykolab/setup/setup_ldap.py:302
-msgid "Enabling attribute uniqueness plugin"
+#: ../pykolab/plugins/__init__.py:144
+#, python-format
+msgid "Not setting runtime for plugin %s: No function 'set_runtime()'"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:308
-msgid "Enabling referential integrity plugin"
+#: ../pykolab/plugins/__init__.py:163 ../pykolab/plugins/__init__.py:165
+#, python-format
+msgid "Cannot add options for plugin %s: %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_ldap.py:314
-msgid "Enabling and configuring account policy plugin"
+#: ../pykolab/plugins/__init__.py:167
+#, python-format
+msgid "Not adding options for plugin %s: No function 'add_options()'"
 msgstr ""
 
-#. TODO: Add kolab-admin role
-#: .././pykolab/setup/setup_ldap.py:329
-msgid "Adding the kolab-admin role"
+#: ../pykolab/plugins/__init__.py:187
+#, python-format
+msgid "Cannot check options for plugin %s: %s"
 msgstr ""
 
-#. TODO: User writeable attributes on root_dn
-#: .././pykolab/setup/setup_ldap.py:340
+#: ../pykolab/plugins/__init__.py:189
 #, python-format
-msgid "Setting access control to %s"
+msgid "Not checking options for plugin %s: No function 'check_options()'"
 msgstr ""
 
-#: .././pykolab/setup/setup_roundcube.py:40
-msgid "Setup Roundcube."
+#: ../pykolab/plugins/__init__.py:227
+#, python-format
+msgid "Executing hook %s for plugin %s"
 msgstr ""
 
-#: .././pykolab/setup/setup_roundcube.py:44
-#: .././pykolab/setup/setup_freebusy.py:49
-msgid "MySQL roundcube password"
+#: ../pykolab/plugins/__init__.py:231 ../pykolab/plugins/__init__.py:233
+#, python-format
+msgid "Cannot execute hook %s for plugin %s: %s"
 msgstr ""
 
-#: .././pykolab/setup/components.py:56
-msgid "Display this help."
+#: ../pykolab/plugins/recipientpolicy/__init__.py:79
+msgid "Attribute substitution for 'mail' failed in Recipient Policy"
 msgstr ""
 
-#: .././pykolab/setup/components.py:122
-msgid "No component selected, continuing for all components"
+#: ../pykolab/plugins/recipientpolicy/__init__.py:115
+msgid "Could not parse the alternative mail routines"
 msgstr ""
 
-#: .././pykolab/setup/components.py:157 .././pykolab/setup/components.py:162
-msgid "No such component."
+#: ../pykolab/plugins/recipientpolicy/__init__.py:127
+#: ../pykolab/plugins/recipientpolicy/__init__.py:137
+msgid ""
+"Attribute substitution for 'alternative_mail' failed in Recipient Policy"
 msgstr ""
 
-#: .././pykolab/setup/setup_mta.py:40
-msgid "Setup MTA."
+#: ../pykolab/setup/components.py:56
+msgid "Display this help."
 msgstr ""
 
-#: .././pykolab/setup/setup_mta.py:220
-#, python-format
-msgid "Setting key %r to %r"
+#: ../pykolab/setup/components.py:122
+msgid "No component selected, continuing for all components"
 msgstr ""
 
-#: .././pykolab/setup/setup_mta.py:248
-msgid "Could not write out Postfix configuration file /etc/postfix/master.cf"
+#: ../pykolab/setup/components.py:157 ../pykolab/setup/components.py:162
+msgid "No such component."
 msgstr ""
 
-#: .././pykolab/setup/setup_imap.py:39
-msgid "Setup IMAP."
+#: ../pykolab/setup/setup_freebusy.py:46
+msgid "Setup Free/Busy."
 msgstr ""
 
-#: .././pykolab/setup/setup_imap.py:82 .././pykolab/setup/setup_imap.py:107
-msgid "Could not write out Cyrus IMAP configuration file /etc/imapd.conf"
+#: ../pykolab/setup/setup_freebusy.py:50
+msgid "Free/Busy is not installed on this system"
 msgstr ""
 
-#: .././pykolab/setup/setup_kolabd.py:38
-msgid "Setup the Kolab daemon."
+#: ../pykolab/setup/setup_freebusy.py:55
+msgid ""
+"\n"
+"                        Please supply the MySQL password for the "
+"'roundcube'\n"
+"                        user. You have supplied this password earlier, and "
+"it is\n"
+"                        available from the database URI setting in\n"
+"                        /etc/roundcubemail/db.inc.php.\n"
+"                    "
 msgstr ""
 
-#: .././pykolab/setup/setup_mysql.py:39
-msgid "Setup MySQL."
+#: ../pykolab/setup/setup_freebusy.py:64
+#: ../pykolab/setup/setup_roundcube.py:56
+msgid "MySQL roundcube password"
 msgstr ""
 
-#: .././pykolab/setup/setup_mysql.py:53
-msgid "MySQL root password"
+#: ../pykolab/setup/setup_freebusy.py:92
+#: ../pykolab/setup/setup_roundcube.py:115 ../pykolab/setup/setup_zpush.py:71
+#, python-format
+msgid "Using template file %r"
 msgstr ""
 
-#: .././pykolab/setup/setup_freebusy.py:40
-msgid "Setup Free/Busy."
+#: ../pykolab/setup/setup_freebusy.py:99
+#: ../pykolab/setup/setup_roundcube.py:122 ../pykolab/setup/setup_zpush.py:78
+#, python-format
+msgid "Successfully compiled template %r, writing out to %r"
 msgstr ""
 
-#: .././pykolab/setup/setup_freebusy.py:44
-msgid "Free/Busy is not installed on this system"
+#: ../pykolab/setup/setup_freebusy.py:119 ../pykolab/setup/setup_imap.py:143
+#: ../pykolab/setup/setup_ldap.py:258 ../pykolab/setup/setup_ldap.py:459
+#: ../pykolab/setup/setup_mta.py:299 ../pykolab/setup/setup_mysql.py:49
+#: ../pykolab/setup/setup_roundcube.py:183 ../pykolab/setup/setup_zpush.py:98
+msgid "Could not start and configure to start on boot, the "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:45
-#, python-format
-msgid "Cleaning up ACL entries for %s across all folders"
+#: ../pykolab/setup/setup_imap.py:44
+msgid "Setup IMAP."
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:60
-#, python-format
-msgid "Cleaning up ACL entries referring to identifier %s"
+#: ../pykolab/setup/setup_imap.py:88 ../pykolab/setup/setup_imap.py:113
+msgid "Could not write out Cyrus IMAP configuration file /etc/imapd.conf"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:69
-#, python-format
-msgid "Iterating over %d folders"
+#: ../pykolab/setup/setup_kolabd.py:43
+msgid "Setup the Kolab daemon."
 msgstr ""
 
-#. Set the ACL to '' (effectively deleting the ACL entry)
-#: .././pykolab/imap/__init__.py:82
+#: ../pykolab/setup/setup_kolabd.py:51
 #, python-format
-msgid "Removing acl %r for subject %r from folder %r"
+msgid ""
+"\n"
+"                            Copying the configuration section for 'example."
+"org' over to\n"
+"                            a section applicable to your domain '%s'.\n"
+"                        "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:143
-#, python-format
-msgid "Logging on to Cyrus IMAP server %s"
+#: ../pykolab/setup/setup_ldap.py:43
+msgid "LDAP Options"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:152
-#, python-format
-msgid "Logging on to Dovecot IMAP server %s"
+#: ../pykolab/setup/setup_ldap.py:50
+msgid "Specify FQDN (overriding defaults)."
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:161
-#, python-format
-msgid "Logging on to generic IMAP server %s"
+#: ../pykolab/setup/setup_ldap.py:54
+msgid "Setup LDAP."
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:179
-#, python-format
-msgid "Reusing existing IMAP server connection to %s"
+#: ../pykolab/setup/setup_ldap.py:66
+msgid ""
+"\n"
+"                        Please supply a password for the LDAP administrator "
+"user\n"
+"                        'admin', used to login to the graphical console of "
+"389\n"
+"                        Directory server.\n"
+"                    "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:181
-#, python-format
-msgid "Reconnecting to IMAP server %s"
+#: ../pykolab/setup/setup_ldap.py:74
+msgid "Administrator password"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:197
-msgid "Called imap.disconnect() on a server that "
+#: ../pykolab/setup/setup_ldap.py:81
+msgid ""
+"\n"
+"                        Please supply a password for the LDAP Directory "
+"Manager\n"
+"                        user, which is the administrator user you will be "
+"using\n"
+"                        to at least initially log in to the Web Admin, and "
+"that\n"
+"                        Kolab uses to perform administrative tasks.\n"
+"                    "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:213
-#, python-format
-msgid "%r has no attribute %s"
+#: ../pykolab/setup/setup_ldap.py:90
+msgid "Directory Manager password"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:221 .././pykolab/imap/__init__.py:239
-#, python-format
-msgid "Creating new shared folder %s"
+#: ../pykolab/setup/setup_ldap.py:96
+msgid "User"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:255
-#, python-format
-msgid "Creating new mailbox for user %s"
+#: ../pykolab/setup/setup_ldap.py:97
+msgid "Group"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:280
-#, python-format
-msgid "Creating additional folders for user %s"
+#: ../pykolab/setup/setup_ldap.py:125
+msgid ""
+"\n"
+"                        This setup procedure plans to set up Kolab Groupware "
+"for\n"
+"                        the following domain name space. This domain name "
+"is\n"
+"                        obtained from the reverse DNS entry on your network\n"
+"                        interface. Please confirm this is the appropriate "
+"domain\n"
+"                        name space.\n"
+"                    "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:302
-#, python-format
-msgid "Mailbox already exists: %s"
+#: ../pykolab/setup/setup_ldap.py:139
+msgid "Domain name to use"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:345
-msgid "Subscribing user to the additional folders"
+#: ../pykolab/setup/setup_ldap.py:144 ../pykolab/setup/setup_ldap.py:169
+msgid ""
+"\n"
+"                                    Invalid input. Please try again.\n"
+"                                "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:380 .././pykolab/imap/__init__.py:459
-#, python-format
-msgid "Renaming INBOX from %s to %s"
+#: ../pykolab/setup/setup_ldap.py:153
+msgid ""
+"\n"
+"                        The standard root dn we composed for you follows. "
+"Please\n"
+"                        confirm this is the root dn you wish to use.\n"
+"                    "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:384
-#, python-format
-msgid "Could not rename INBOX folder %s to %s"
+#: ../pykolab/setup/setup_ldap.py:164
+msgid "Root DN to use"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:386 .././pykolab/imap/__init__.py:463
-#, python-format
-msgid "Moving INBOX folder %s won't succeed as target folder %s already exists"
+#: ../pykolab/setup/setup_ldap.py:213
+msgid "Setting up 389 Directory Server"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:399
-#, python-format
-msgid "Looking for folder '%s', we found folders: %r"
+#: ../pykolab/setup/setup_ldap.py:223
+msgid "Setup DS stdout:"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:422
-#, python-format
-msgid "Setting ACL rights %s for subject %s on folder "
+#: ../pykolab/setup/setup_ldap.py:226
+msgid "Setup DS stderr:"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:433
-#, python-format
-msgid "Removing ACL rights %s for subject %s on folder "
+#: ../pykolab/setup/setup_ldap.py:249 ../pykolab/setup/setup_mysql.py:129
+msgid "Could not find the Kolab schema file"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:456
-#, python-format
-msgid "Found old INBOX folder %s"
+#: ../pykolab/setup/setup_ldap.py:263
+msgid ""
+"\n"
+"                        Please supply a Cyrus Administrator password. This\n"
+"                        password is used by Kolab to execute administrative\n"
+"                        tasks in Cyrus IMAP. You may also need the password\n"
+"                        yourself to troubleshoot Cyrus IMAP and/or perform\n"
+"                        other administrative tasks against Cyrus IMAP "
+"directly.\n"
+"                    "
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:465
-#, python-format
-msgid "Did not find old folder user/%s to rename"
+#: ../pykolab/setup/setup_ldap.py:273
+msgid "Cyrus Administrator password"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:467
-msgid "Value for user is not a dictionary"
+#: ../pykolab/setup/setup_ldap.py:280
+msgid ""
+"\n"
+"                        Please supply a Kolab Service account password. "
+"This\n"
+"                        account is used by various services such as "
+"Postfix,\n"
+"                        and Roundcube, as anonymous binds to the LDAP "
+"server\n"
+"                        will not be allowed.\n"
+"                    "
 msgstr ""
 
-#. TODO: Go in fact correct the quota.
-#: .././pykolab/imap/__init__.py:527
-#, python-format
-msgid "Cannot get current IMAP quota for folder %s"
+#: ../pykolab/setup/setup_ldap.py:289
+msgid "Kolab Service password"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:540
-#, python-format
-msgid "Quota for %s currently is %s"
+#: ../pykolab/setup/setup_ldap.py:299
+msgid "Writing out configuration to kolab.conf"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:546
-#, python-format
-msgid "Adjusting authentication database quota for folder %s to %d"
+#: ../pykolab/setup/setup_ldap.py:313
+msgid "Inserting service users into LDAP."
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:551
-#, python-format
-msgid "Correcting quota for %s to %s (currently %s)"
+#: ../pykolab/setup/setup_ldap.py:357
+msgid "Writing out cn=kolab,cn=config"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:633
+#. TODO: Add kolab-admin role
+#. TODO: Assign kolab-admin admin ACLs
+#: ../pykolab/setup/setup_ldap.py:381
 #, python-format
-msgid "Checking folder: %s"
+msgid "Adding domain %s to list of domains for this deployment"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:638
-#, python-format
-msgid "Folder has no corresponding user (1): %s"
+#: ../pykolab/setup/setup_ldap.py:395
+msgid "Disabling anonymous binds"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:641
-#, python-format
-msgid "Folder has no corresponding user (2): %s"
+#. TODO: Ensure the uid attribute is unique
+#. TODO^2: Consider renaming the general "attribute uniqueness to "uid attribute uniqueness"
+#: ../pykolab/setup/setup_ldap.py:403
+msgid "Enabling attribute uniqueness plugin"
 msgstr ""
 
-#. We got user identifier only
-#: .././pykolab/imap/__init__.py:656
-msgid "Please don't give us just a user identifier"
+#: ../pykolab/setup/setup_ldap.py:409
+msgid "Enabling referential integrity plugin"
 msgstr ""
 
-#: .././pykolab/imap/__init__.py:659
-#, python-format
-msgid "Deleting folder %s"
+#: ../pykolab/setup/setup_ldap.py:415
+msgid "Enabling and configuring account policy plugin"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:78
-#, python-format
-msgid "Could not connect to Cyrus IMAP server %r"
+#. TODO: Add kolab-admin role
+#: ../pykolab/setup/setup_ldap.py:430
+msgid "Adding the kolab-admin role"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:101
+#. TODO: User writeable attributes on root_dn
+#: ../pykolab/setup/setup_ldap.py:441
 #, python-format
-msgid "Continuing with separator: %r"
-msgstr ""
-
-#: .././pykolab/imap/cyrus.py:106
-msgid "Detected we are running in a Murder topology"
+msgid "Setting access control to %s"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:110
-msgid "This system is not part of a murder topology"
+#: ../pykolab/setup/setup_mta.py:40
+msgid "Setup MTA."
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:131
+#: ../pykolab/setup/setup_mta.py:221
 #, python-format
-msgid "Checking actual backend server for folder %s through annotations"
+msgid "Setting key %r to %r"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:146
-#, python-format
-msgid "Could not get the annotations after %s tries."
+#: ../pykolab/setup/setup_mta.py:249
+msgid "Could not write out Postfix configuration file /etc/postfix/master.cf"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:150
-#, python-format
-msgid "No annotations for %s: %r"
+#: ../pykolab/setup/setup_mta.py:281
+msgid "Could not write out Amavis configuration file /etc/amavisd/amavisd.conf"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:157
-#, python-format
-msgid "Server for INBOX folder %s is %s"
+#: ../pykolab/setup/setup_mysql.py:39
+msgid "Setup MySQL."
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:169
-#, python-format
-msgid "Setting quota for INBOX folder %s to %s"
+#: ../pykolab/setup/setup_mysql.py:53
+msgid ""
+"\n"
+"                    Please supply a root password for MySQL. This password "
+"will\n"
+"                    be the administrative user for this MySQL server, and "
+"it\n"
+"                    should be kept a secret. After this setup process has\n"
+"                    completed, Kolab is going to discard and forget about "
+"this\n"
+"                    password, but you will need it for administrative tasks "
+"in\n"
+"                    MySQL.\n"
+"                "
+msgstr ""
+
+#: ../pykolab/setup/setup_mysql.py:64
+msgid "MySQL root password"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:173
-#, python-format
-msgid "Could not set quota for mailfolder %s"
+#: ../pykolab/setup/setup_mysql.py:103
+msgid ""
+"\n"
+"                        Please supply a password for the MySQL user "
+"'kolab'.\n"
+"                        This password will be used by Kolab services, such "
+"as\n"
+"                        the Web Administration Panel.\n"
+"                    "
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:182
-#, python-format
-msgid "Moving INBOX folder %s to %s"
+#: ../pykolab/setup/setup_mysql.py:111
+msgid "MySQL kolab password"
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:194
-#, python-format
-msgid "Setting annotation %s on folder %s"
+#: ../pykolab/setup/setup_roundcube.py:44
+msgid "Setup Roundcube."
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:201
-#, python-format
-msgid "Could not set annotation %r on mail folder %r: %r"
+#: ../pykolab/setup/setup_roundcube.py:48
+msgid ""
+"\n"
+"                    Please supply a password for the MySQL user "
+"'roundcube'.\n"
+"                    This password will be used by the Roundcube webmail\n"
+"                    interface.\n"
+"                "
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:205
-#, python-format
-msgid "Transferring folder %s from %s to %s"
+#: ../pykolab/setup/setup_zpush.py:41
+msgid "Setup zpush."
 msgstr ""
 
-#: .././pykolab/imap/cyrus.py:265
-#, python-format
-msgid "Undeleting %s to %s"
+#: ../pykolab/setup/setup_zpush.py:45
+msgid "Z-Push is not installed on this system"
 msgstr ""
 
 #. start_max = (int)(time.time())
-#: .././pykolab/telemetry.py:588
+#: ../pykolab/telemetry.py:588
 #, python-format
 msgid "Expiring sessions that started before or on %d"
 msgstr ""
 
-#: .././pykolab/telemetry.py:601
+#: ../pykolab/telemetry.py:601
 #, python-format
 msgid "Expiring session ID: %d"
 msgstr ""
 
-#: .././pykolab/telemetry.py:620
+#: ../pykolab/telemetry.py:620
 #, python-format
 msgid "Session with ID %d expired from database"
 msgstr ""
 
-#: .././pykolab/telemetry.py:646
+#: ../pykolab/telemetry.py:646
 #, python-format
 msgid "Operational Error in telemetry database: %s"
 msgstr ""
 
-#: .././pykolab/tests.old/mail.py:92
-#, python-format
-msgid "Creating %d Mails"
+#: ../pykolab/telemetry.py:652
+msgid "No database available"
 msgstr ""
 
-#: .././pykolab/tests.old/mail.py:116
+#: ../pykolab/tests/imap/test_login.py:44
 #, python-format
-msgid "Sending message %s through SMTP targeting user %s@%s"
+msgid "Connecting at %s"
 msgstr ""
 
-#: .././pykolab/tests.old/mail.py:119
+#: ../pykolab/tests/imap/test_login.py:46
 #, python-format
-msgid "Sending message %s through LMTP targeting user %s@%s"
+msgid "Connected at %s"
 msgstr ""
 
-#: .././pykolab/tests.old/mail.py:122
+#: ../pykolab/tests/imap/test_login.py:51
 #, python-format
-msgid "Saving message %s to IMAP (user %s, folder %s)"
+msgid "Logging in at %s"
 msgstr ""
 
-#: .././pykolab/tests.old/mail.py:125 .././pykolab/tests.old/calendar.py:198
-#: .././pykolab/tests.old/contacts.py:145
-msgid "Somehow ended up NOT sending these messages"
+#: ../pykolab/tests/imap/test_login.py:53
+#, python-format
+msgid "Logged in at %s"
 msgstr ""
 
-#: .././pykolab/tests.old/calendar.py:146
+#: ../pykolab/tests/imap/test_login.py:60
 #, python-format
-msgid "Creating %d Events"
+msgid "Test failure in %s"
 msgstr ""
 
-#: .././pykolab/tests.old/calendar.py:177
-#, python-format
-msgid "Creating Calendar item number %d"
+#: ../pykolab/tests/__init__.py:39
+msgid "Test Options"
 msgstr ""
 
-#: .././pykolab/tests.old/calendar.py:189
-#: .././pykolab/tests.old/contacts.py:136
-#, python-format
-msgid "Sending UID message %s through SMTP targeting user %s@%s"
+#: ../pykolab/tests/__init__.py:45
+msgid "Run tests in suite SUITE. Implies a certain set of items being tested."
 msgstr ""
 
-#: .././pykolab/tests.old/calendar.py:192
-#: .././pykolab/tests.old/contacts.py:139
-#, python-format
-msgid "Sending UID message %s through LMTP targeting user %s@%s"
+#: ../pykolab/tests/tests.py:53
+msgid "List tests"
 msgstr ""
 
-#: .././pykolab/tests.old/calendar.py:195
-#: .././pykolab/tests.old/contacts.py:142
+#. This is a nested test
+#: ../pykolab/tests/tests.py:86
 #, python-format
-msgid "Saving UID message %s to IMAP (user %s, folder %s)"
+msgid "Test Suite: %s"
 msgstr ""
 
-#: .././pykolab/tests.old/zpush/test_000_001.py:47
-#, python-format
-msgid "Could not load %sItem from %s, skipping the testing."
+#: ../pykolab/tests/tests.py:100 ../pykolab/tests/tests.py:105
+msgid "No such test."
 msgstr ""
 
-#: .././pykolab/tests.old/zpush/test_000_000.py:77
+#: ../pykolab/tests/tests.py:157 ../pykolab/tests/tests.py:161
 #, python-format
-msgid "Authentication failure for %s"
+msgid "Test '%s' already registered"
 msgstr ""
 
-#: .././pykolab/tests.old/contacts.py:105
+#: ../pykolab/utils.py:55 ../pykolab/utils.py:57
 #, python-format
-msgid "Creating %d Contacts"
+msgid "Confirm %s: "
 msgstr ""
 
-#: .././pykolab/tests.old/contacts.py:124
-#, python-format
-msgid "Creating Contact item number %d"
+#: ../pykolab/utils.py:60
+msgid "Incorrect confirmation. "
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:207
+#: ../pykolab/utils.py:65 ../pykolab/utils.py:70
 #, python-format
-msgid "Adding policy request to instance %s"
+msgid "%s: "
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:442
-msgid "Unauthorized access not allowed"
+#: ../pykolab/utils.py:67 ../pykolab/utils.py:72
+#, python-format
+msgid "%s [%s]: "
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:481
-#: .././bin/kolab_smtp_access_policy.py:541
-#, python-format
-msgid "Could not find envelope sender user %s"
+#: ../pykolab/utils.py:117
+msgid "Please answer 'yes' or 'no'."
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:568
-#, python-format
-msgid "%s is unauthorized to send on behalf of %s"
+#: ../saslauthd/__init__.py:94 ../saslauthd/__init__.py:102
+msgid "Traceback occurred, please report a bug at http://bugzilla.kolabsys.com"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:578
-#, python-format
-msgid "User %s attempted to use envelope sender address %s "
+#: ../wallace/__init__.py:52
+msgid "Bind address for Wallace."
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:638
-#: .././bin/kolab_smtp_access_policy.py:649
-#, python-format
-msgid "Found user %s to be a delegate user of %s"
+#: ../wallace/__init__.py:78
+msgid "Port that Wallace is supposed to use."
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:673
+#: ../wallace/__init__.py:122 ../wallace/__init__.py:146
+#: ../wallace/__init__.py:350 ../wallace/__init__.py:378
 #, python-format
-msgid "Verifying authenticated sender '%(sender)s' with "
+msgid "Number of threads currently running: %d"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:678
+#: ../wallace/__init__.py:131 ../wallace/__init__.py:155
 #, python-format
-msgid "Verifying unauthenticated sender '%(sender)s' "
+msgid "Continuing with %d threads currently running"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:695
+#. TODO: Apply throttling
+#: ../wallace/__init__.py:138
 #, python-format
-msgid "Reproducing verify_recipient(%s, %s) from "
+msgid "Creating thread for message in %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:712
+#: ../wallace/__init__.py:162
 #, python-format
-msgid "Using authentication domain %s instead of %s"
+msgid "Running thread %s for message file %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:722
+#: ../wallace/__init__.py:171
 #, python-format
-msgid "Domain %s is a primary domain"
+msgid "This message was already in module %s, delegating "
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:730
+#: ../wallace/__init__.py:180
 #, python-format
-msgid "Checking the recipient for domain %s that is not "
+msgid "It was also in a certain stage: %s, letting "
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:792
-msgid "Could not find this user, accepting"
+#: ../wallace/__init__.py:188 ../wallace/__init__.py:194
+#: ../wallace/__init__.py:204
+#, python-format
+msgid "Executing module %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:862
-#: .././bin/kolab_smtp_access_policy.py:915
-#, python-format
-msgid "Sender %s is not allowed to send to "
+#: ../wallace/__init__.py:218
+msgid "Picking up deferred messages for wallace"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:902
+#: ../wallace/__init__.py:238
 #, python-format
-msgid "Reproducing verify_sender(%r) from cache, "
+msgid "Picking up deferred messages for module %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:931
-msgid "Sender uses unauthorized envelope sender address"
+#. Sleep for 300 seconds before reprocessing the deferred queues.
+#. TODO: Consider using queue_run_delay from Postfix, which is where
+#. the default value of 300 seconds comes from.
+#: ../wallace/__init__.py:262
+msgid "Sleeping for 300 seconds"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:948
-msgid "Could not verify sender"
+#: ../wallace/__init__.py:287
+#, python-format
+msgid "Could not bind to socket on port %d on bind "
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:955
-msgid "Verifying whether sender is allowed to send to "
+#: ../wallace/__init__.py:297
+msgid "Could not shut down socket"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:969
+#: ../wallace/__init__.py:371
 #, python-format
-msgid "Result is %r"
+msgid "Picking up spooled email file %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:974
-msgid "No recipient policy restrictions exist for this sender"
+#: ../wallace/__init__.py:401
+msgid "Accepted connection"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:983
-msgid "Found a recipient policy to apply for this sender."
+#: ../wallace/module_optout.py:57
+#, python-format
+msgid "Issuing callback after processing to stage %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:998
+#: ../wallace/module_optout.py:58
 #, python-format
-msgid "Sender %s not allowed to send to recipient "
+msgid "Testing cb_action_%s()"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1029
-msgid "Cleaning up the cache"
+#: ../wallace/module_optout.py:60
+#, python-format
+msgid "Attempting to execute cb_action_%s()"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1066
+#. modules.next_module('optout')
+#: ../wallace/module_optout.py:66
 #, python-format
-msgid "Operational Error in caching: %s"
+msgid "Consulting opt-out service for %r, %r"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1117
+#: ../wallace/module_optout.py:92
 #, python-format
-msgid "Caching the policy result with timestamp %d"
+msgid "Running opt-out consult from envelope sender '%s "
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1194
+#: ../wallace/module_optout.py:156
 #, python-format
-msgid "Returning action DEFER_IF_PERMIT: %s"
+msgid "Attempting to execute cb_action_%s(%r, %r)"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1199
+#: ../wallace/module_optout.py:177
 #, python-format
-msgid "Returning action DUNNO: %s"
+msgid "Could not send request to optout_url %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1204
+#. This is a nested module
+#: ../wallace/modules.py:96
 #, python-format
-msgid "Returning action HOLD: %s"
+msgid "Module Group: %s"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1209
-#, python-format
-msgid "Returning action PERMIT: %s"
+#: ../wallace/modules.py:107 ../wallace/modules.py:112
+msgid "No such module."
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1214
+#: ../wallace/modules.py:118
 #, python-format
-msgid "Returning action REJECT: %s"
+msgid "Holding message in queue for manual review (%s by %s)"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1258
-msgid "Starting to loop for new request"
+#: ../wallace/modules.py:121
+#, python-format
+msgid "Deferring message in %s (by module %s)"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1267
-msgid "End of current request"
+#: ../wallace/modules.py:131
+#, python-format
+msgid "The time when the message was sent: %r"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1271
+#: ../wallace/modules.py:132
 #, python-format
-msgid "Getting line: %s"
+msgid "The time now: %r"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1275
-msgid "Returning request"
+#: ../wallace/modules.py:133
+#, python-format
+msgid "The time delta: %r"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1332
+#. TODO: Send NDR back to user
+#: ../wallace/modules.py:137
 #, python-format
-msgid "Got request instance %s"
+msgid "Message in file %s older then 5 days, deleting"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1341
+#: ../wallace/modules.py:162
 #, python-format
-msgid "Request instance %s is in state %s"
+msgid "Rejecting message in %s (by module %s)"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1349
+#: ../wallace/modules.py:180
 #, python-format
-msgid "Request instance %s is not yet in DATA state"
+msgid ""
+"This is the email system Wallace at %s.\n"
+"\n"
+"I'm sorry to inform you we could not deliver the attached message\n"
+"to the following recipients:\n"
+"\n"
+"- %s\n"
+"\n"
+"Your message is being delivered to any other recipients you may have\n"
+"sent your message to. There is no need to resend the message to those\n"
+"recipients.\n"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1361
+#: ../wallace/modules.py:195
 #, python-format
-msgid "Request instance %s reached DATA state"
+msgid ""
+"X-Wallace-Module: %s\n"
+"X-Wallace-Result: REJECT\n"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1379
-msgid "Sender access denied"
+#: ../wallace/modules.py:248
+#, python-format
+msgid "Accepting message in %s (by module %s)"
 msgstr ""
 
-#: .././bin/kolab_smtp_access_policy.py:1381
-msgid "Recipient access denied"
+#: ../wallace/modules.py:316
+#, python-format
+msgid "Module '%s' already registered"
 msgstr ""
